<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:42 EST 2015 -->
<TITLE>
GenotypeConcordance (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="GenotypeConcordance (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/GenotypeConcordance.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/vcf/GatherVcfs.html" title="class in picard.vcf"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/vcf/GenotypeConcordance.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="GenotypeConcordance.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.vcf</FONT>
<BR>
Class GenotypeConcordance</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../resources/inherit.gif" ALT="extended by "><A HREF="../../picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">picard.cmdline.CommandLineProgram</A>
      <IMG SRC="../../resources/inherit.gif" ALT="extended by "><B>picard.vcf.GenotypeConcordance</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>GenotypeConcordance</B><DT>extends <A HREF="../../picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</A></DL>
</PRE>

<P>
Calculates the concordance between genotype data for two samples in two different VCFs - one being considered the truth (or reference)
 the other being the call.  The concordance is broken into separate results sections for SNPs and indels.  Summary and detailed statistics
 are reported
<P>

<P>
<DL>
<DT><B>Author:</B></DT>
  <DD>Tim Fennell, George Grant</DD>
</DL>
<HR>

<P>
<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.lang.String</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#CALL_SAMPLE">CALL_SAMPLE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.io.File</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#CALL_VCF">CALL_VCF</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;java.lang.String</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#CONTINGENCY_METRICS_FILE_EXTENSION">CONTINGENCY_METRICS_FILE_EXTENSION</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;java.lang.String</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#DETAILED_METRICS_FILE_EXTENSION">DETAILED_METRICS_FILE_EXTENSION</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;<A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A></CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#indelCounter">indelCounter</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#INTERSECT_INTERVALS">INTERSECT_INTERVALS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.util.List&lt;java.io.File&gt;</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#INTERVALS">INTERVALS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;int</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#MIN_DP">MIN_DP</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;int</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#MIN_GQ">MIN_GQ</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#MISSING_SITES_HOM_REF">MISSING_SITES_HOM_REF</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.io.File</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#OUTPUT">OUTPUT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#OUTPUT_ALL_ROWS">OUTPUT_ALL_ROWS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;<A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A></CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#snpCounter">snpCounter</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;java.lang.String</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#SUMMARY_METRICS_FILE_EXTENSION">SUMMARY_METRICS_FILE_EXTENSION</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.lang.String</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#TRUTH_SAMPLE">TRUTH_SAMPLE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.io.File</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#TRUTH_VCF">TRUTH_VCF</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#USE_VCF_INDEX">USE_VCF_INDEX</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;<A NAME="fields_inherited_from_class_picard.cmdline.CommandLineProgram"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Fields inherited from class picard.cmdline.<A HREF="../../picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../picard/cmdline/CommandLineProgram.html#COMPRESSION_LEVEL">COMPRESSION_LEVEL</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#CREATE_INDEX">CREATE_INDEX</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#CREATE_MD5_FILE">CREATE_MD5_FILE</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#GA4GH_CLIENT_SECRETS">GA4GH_CLIENT_SECRETS</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#MAX_RECORDS_IN_RAM">MAX_RECORDS_IN_RAM</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#QUIET">QUIET</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#REFERENCE_SEQUENCE">REFERENCE_SEQUENCE</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#TMP_DIR">TMP_DIR</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#VALIDATION_STRINGENCY">VALIDATION_STRINGENCY</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#VERBOSITY">VERBOSITY</A></CODE></TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#GenotypeConcordance()">GenotypeConcordance</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;java.lang.String[]</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#customCommandLineValidation()">customCommandLineValidation</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Put any custom command-line validation in an override of this method.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;int</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#doWork()">doWork</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do the work after command line has been parsed.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;<A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A></CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#getIndelCounter()">getIndelCounter</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;<A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A></CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#getSnpCounter()">getSnpCounter</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/vcf/GenotypeConcordance.html#main(java.lang.String[])">main</A></B>(java.lang.String[]&nbsp;args)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_picard.cmdline.CommandLineProgram"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class picard.cmdline.<A HREF="../../picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../picard/cmdline/CommandLineProgram.html#getCommandLine()">getCommandLine</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getCommandLineParser()">getCommandLineParser</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getDefaultHeaders()">getDefaultHeaders</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getMetricsFile()">getMetricsFile</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getNestedOptions()">getNestedOptions</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getNestedOptionsForHelp()">getNestedOptionsForHelp</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getStandardUsagePreamble()">getStandardUsagePreamble</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#getVersion()">getVersion</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#instanceMain(java.lang.String[])">instanceMain</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#instanceMainWithExit(java.lang.String[])">instanceMainWithExit</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#parseArgs(java.lang.String[])">parseArgs</A>, <A HREF="../../picard/cmdline/CommandLineProgram.html#setDefaultHeaders(java.util.List)">setDefaultHeaders</A></CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, equals, finalize, getClass, hashCode, notify, notifyAll, toString, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ============ FIELD DETAIL =========== -->

<A NAME="field_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Field Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="TRUTH_VCF"><!-- --></A><H3>
TRUTH_VCF</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#shortName()">shortName</A>="TV",
        <A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="The VCF containing the truth sample")
</FONT>public java.io.File <B>TRUTH_VCF</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="CALL_VCF"><!-- --></A><H3>
CALL_VCF</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#shortName()">shortName</A>="CV",
        <A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="The VCF containing the call sample")
</FONT>public java.io.File <B>CALL_VCF</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="OUTPUT"><!-- --></A><H3>
OUTPUT</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#shortName()">shortName</A>="O",
        <A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="Basename for the two metrics files that are to be written. Resulting files will be <OUTPUT>.genotype_concordance_summary_metrics  and <OUTPUT>.genotype_concordance_detail_metrics.")
</FONT>public java.io.File <B>OUTPUT</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="TRUTH_SAMPLE"><!-- --></A><H3>
TRUTH_SAMPLE</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#shortName()">shortName</A>="TS",
        <A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="The name of the truth sample within the truth VCF")
</FONT>public java.lang.String <B>TRUTH_SAMPLE</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="CALL_SAMPLE"><!-- --></A><H3>
CALL_SAMPLE</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#shortName()">shortName</A>="CS",
        <A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="The name of the call sample within the call VCF")
</FONT>public java.lang.String <B>CALL_SAMPLE</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="INTERVALS"><!-- --></A><H3>
INTERVALS</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="One or more interval list files that will be used to limit the genotype concordance.  Note - if intervals are specified, the VCF files must be indexed.")
</FONT>public java.util.List&lt;java.io.File&gt; <B>INTERVALS</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="INTERSECT_INTERVALS"><!-- --></A><H3>
INTERSECT_INTERVALS</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="If true, multiple interval lists will be intersected. If false multiple lists will be unioned.")
</FONT>public boolean <B>INTERSECT_INTERVALS</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="MIN_GQ"><!-- --></A><H3>
MIN_GQ</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="Genotypes below this genotype quality will have genotypes classified as LowGq.")
</FONT>public int <B>MIN_GQ</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="MIN_DP"><!-- --></A><H3>
MIN_DP</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="Genotypes below this depth will have genotypes classified as LowDp.")
</FONT>public int <B>MIN_DP</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="OUTPUT_ALL_ROWS"><!-- --></A><H3>
OUTPUT_ALL_ROWS</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="If true, output all rows in detailed statistics even when count == 0.  When false only output rows with non-zero counts.")
</FONT>public boolean <B>OUTPUT_ALL_ROWS</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="USE_VCF_INDEX"><!-- --></A><H3>
USE_VCF_INDEX</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="If true, use the VCF index, else iterate over the entire VCF.",
        <A HREF="../../picard/cmdline/Option.html#optional()">optional</A>=true)
</FONT>public boolean <B>USE_VCF_INDEX</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="MISSING_SITES_HOM_REF"><!-- --></A><H3>
MISSING_SITES_HOM_REF</H3>
<PRE>
<FONT SIZE="-1"><A HREF="../../picard/cmdline/Option.html" title="annotation in picard.cmdline">@Option</A>(<A HREF="../../picard/cmdline/Option.html#shortName()">shortName</A>="MISSING_HOM",
        <A HREF="../../picard/cmdline/Option.html#doc()">doc</A>="Default is false, which follows the GA4GH Scheme. If true, missing sites in the truth set will be treated as HOM_REF sites and sites missing in both the truth and call sets will be true negatives. Useful when hom ref sites are left out of the truth set. This flag can only be used with a high confidence interval list.")
</FONT>public boolean <B>MISSING_SITES_HOM_REF</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="SUMMARY_METRICS_FILE_EXTENSION"><!-- --></A><H3>
SUMMARY_METRICS_FILE_EXTENSION</H3>
<PRE>
public static final java.lang.String <B>SUMMARY_METRICS_FILE_EXTENSION</B></PRE>
<DL>
<DL>
<DT><B>See Also:</B><DD><A HREF="../../constant-values.html#picard.vcf.GenotypeConcordance.SUMMARY_METRICS_FILE_EXTENSION">Constant Field Values</A></DL>
</DL>
<HR>

<A NAME="DETAILED_METRICS_FILE_EXTENSION"><!-- --></A><H3>
DETAILED_METRICS_FILE_EXTENSION</H3>
<PRE>
public static final java.lang.String <B>DETAILED_METRICS_FILE_EXTENSION</B></PRE>
<DL>
<DL>
<DT><B>See Also:</B><DD><A HREF="../../constant-values.html#picard.vcf.GenotypeConcordance.DETAILED_METRICS_FILE_EXTENSION">Constant Field Values</A></DL>
</DL>
<HR>

<A NAME="CONTINGENCY_METRICS_FILE_EXTENSION"><!-- --></A><H3>
CONTINGENCY_METRICS_FILE_EXTENSION</H3>
<PRE>
public static final java.lang.String <B>CONTINGENCY_METRICS_FILE_EXTENSION</B></PRE>
<DL>
<DL>
<DT><B>See Also:</B><DD><A HREF="../../constant-values.html#picard.vcf.GenotypeConcordance.CONTINGENCY_METRICS_FILE_EXTENSION">Constant Field Values</A></DL>
</DL>
<HR>

<A NAME="snpCounter"><!-- --></A><H3>
snpCounter</H3>
<PRE>
protected <A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A> <B>snpCounter</B></PRE>
<DL>
<DL>
</DL>
</DL>
<HR>

<A NAME="indelCounter"><!-- --></A><H3>
indelCounter</H3>
<PRE>
protected <A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A> <B>indelCounter</B></PRE>
<DL>
<DL>
</DL>
</DL>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="GenotypeConcordance()"><!-- --></A><H3>
GenotypeConcordance</H3>
<PRE>
public <B>GenotypeConcordance</B>()</PRE>
<DL>
</DL>

<!-- ============ METHOD DETAIL ========== -->

<A NAME="method_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Method Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="getSnpCounter()"><!-- --></A><H3>
getSnpCounter</H3>
<PRE>
public <A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A> <B>getSnpCounter</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="getIndelCounter()"><!-- --></A><H3>
getIndelCounter</H3>
<PRE>
public <A HREF="../../picard/vcf/GenotypeConcordanceCounts.html" title="class in picard.vcf">GenotypeConcordanceCounts</A> <B>getIndelCounter</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="main(java.lang.String[])"><!-- --></A><H3>
main</H3>
<PRE>
public static void <B>main</B>(java.lang.String[]&nbsp;args)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="customCommandLineValidation()"><!-- --></A><H3>
customCommandLineValidation</H3>
<PRE>
protected java.lang.String[] <B>customCommandLineValidation</B>()</PRE>
<DL>
<DD><B>Description copied from class: <CODE><A HREF="../../picard/cmdline/CommandLineProgram.html#customCommandLineValidation()">CommandLineProgram</A></CODE></B></DD>
<DD>Put any custom command-line validation in an override of this method.
 clp is initialized at this point and can be used to print usage and access argv.
 Any options set by command-line parser can be validated.
<P>
<DD><DL>
<DT><B>Overrides:</B><DD><CODE><A HREF="../../picard/cmdline/CommandLineProgram.html#customCommandLineValidation()">customCommandLineValidation</A></CODE> in class <CODE><A HREF="../../picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</A></CODE></DL>
</DD>
<DD><DL>

<DT><B>Returns:</B><DD>null if command line is valid.  If command line is invalid, returns an array of error message
 to be written to the appropriate place.</DL>
</DD>
</DL>
<HR>

<A NAME="doWork()"><!-- --></A><H3>
doWork</H3>
<PRE>
protected int <B>doWork</B>()</PRE>
<DL>
<DD><B>Description copied from class: <CODE><A HREF="../../picard/cmdline/CommandLineProgram.html#doWork()">CommandLineProgram</A></CODE></B></DD>
<DD>Do the work after command line has been parsed. RuntimeException may be
 thrown by this method, and are reported appropriately.
<P>
<DD><DL>
<DT><B>Specified by:</B><DD><CODE><A HREF="../../picard/cmdline/CommandLineProgram.html#doWork()">doWork</A></CODE> in class <CODE><A HREF="../../picard/cmdline/CommandLineProgram.html" title="class in picard.cmdline">CommandLineProgram</A></CODE></DL>
</DD>
<DD><DL>

<DT><B>Returns:</B><DD>program exit status.</DL>
</DD>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/GenotypeConcordance.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/vcf/GatherVcfs.html" title="class in picard.vcf"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/vcf/GenotypeConcordanceContingencyMetrics.html" title="class in picard.vcf"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/vcf/GenotypeConcordance.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="GenotypeConcordance.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
