<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:40 EST 2015 -->
<TITLE>
HsMetrics (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="HsMetrics (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/HsMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../../picard/analysis/directed/HsMetricCollector.html" title="class in picard.analysis.directed"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../../picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../../index.html?picard/analysis/directed/HsMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="HsMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.analysis.directed</FONT>
<BR>
Class HsMetrics</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../../resources/inherit.gif" ALT="extended by ">htsjdk.samtools.metrics.MetricBase
      <IMG SRC="../../../resources/inherit.gif" ALT="extended by "><A HREF="../../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">picard.metrics.MultilevelMetrics</A>
          <IMG SRC="../../../resources/inherit.gif" ALT="extended by "><B>picard.analysis.directed.HsMetrics</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>HsMetrics</B><DT>extends <A HREF="../../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</A></DL>
</PRE>

<P>
The set of metrics captured that are specific to a hybrid selection analysis.
<P>

<P>
<DL>
<DT><B>Author:</B></DT>
  <DD>Tim Fennell</DD>
</DL>
<HR>

<P>
<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#AT_DROPOUT">AT_DROPOUT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A measure of how undercovered <= 50% GC regions are relative to the mean.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#BAIT_DESIGN_EFFICIENCY">BAIT_DESIGN_EFFICIENCY</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Target terrirtoy / bait territory.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.lang.String</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#BAIT_SET">BAIT_SET</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The name of the bait set used in the hybrid selection.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#BAIT_TERRITORY">BAIT_TERRITORY</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of bases which have one or more baits on top of them.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#FOLD_80_BASE_PENALTY">FOLD_80_BASE_PENALTY</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#FOLD_ENRICHMENT">FOLD_ENRICHMENT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fold by which the baited region has been amplified above genomic background.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#GC_DROPOUT">GC_DROPOUT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A measure of how undercovered >= 50% GC regions are relative to the mean.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#GENOME_SIZE">GENOME_SIZE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of bases in the reference genome used for alignment.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.lang.Long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_LIBRARY_SIZE">HS_LIBRARY_SIZE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The estimated number of unique molecules in the selected part of the library.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_PENALTY_100X">HS_PENALTY_100X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The "hybrid selection penalty" incurred to get 80% of target bases to 100X.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_PENALTY_10X">HS_PENALTY_10X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The "hybrid selection penalty" incurred to get 80% of target bases to 10X.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_PENALTY_20X">HS_PENALTY_20X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The "hybrid selection penalty" incurred to get 80% of target bases to 20X.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_PENALTY_30X">HS_PENALTY_30X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The "hybrid selection penalty" incurred to get 80% of target bases to 30X.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_PENALTY_40X">HS_PENALTY_40X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The "hybrid selection penalty" incurred to get 80% of target bases to 40X.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HS_PENALTY_50X">HS_PENALTY_50X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The "hybrid selection penalty" incurred to get 80% of target bases to 50X.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#MEAN_BAIT_COVERAGE">MEAN_BAIT_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The mean coverage of all baits in the experiment.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#MEAN_TARGET_COVERAGE">MEAN_TARGET_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The mean coverage of targets that received at least coverage depth = 2 at one base.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#NEAR_BAIT_BASES">NEAR_BAIT_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF aligned bases that mapped to within a fixed interval of a baited region, but not on a baited region.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#OFF_BAIT_BASES">OFF_BAIT_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF aligned bases that mapped to neither on or near a bait.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#ON_BAIT_BASES">ON_BAIT_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF aligned bases that mapped to a baited region of the genome.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#ON_BAIT_VS_SELECTED">ON_BAIT_VS_SELECTED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of on+near bait bases that are on as opposed to near.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#ON_TARGET_BASES">ON_TARGET_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF aligned bases that mapped to a targeted region of the genome.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_OFF_BAIT">PCT_OFF_BAIT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of aligned PF bases that mapped neither on or near a bait.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_PF_READS">PCT_PF_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PF reads / total reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_PF_UQ_READS">PCT_PF_UQ_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PF Unique Reads / Total Reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_PF_UQ_READS_ALIGNED">PCT_PF_UQ_READS_ALIGNED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PF Reads Aligned / PF Reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_SELECTED_BASES">PCT_SELECTED_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;On+Near Bait Bases / PF Bases Aligned.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_100X">PCT_TARGET_BASES_100X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 100X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_10X">PCT_TARGET_BASES_10X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 10X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_20X">PCT_TARGET_BASES_20X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 20X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_2X">PCT_TARGET_BASES_2X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 2X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_30X">PCT_TARGET_BASES_30X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 30X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_40X">PCT_TARGET_BASES_40X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 40X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_TARGET_BASES_50X">PCT_TARGET_BASES_50X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of ALL target bases achieving 50X or greater coverage.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_USABLE_BASES_ON_BAIT">PCT_USABLE_BASES_ON_BAIT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of aligned, de-duped, on-bait bases out of the PF bases available.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PCT_USABLE_BASES_ON_TARGET">PCT_USABLE_BASES_ON_TARGET</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of aligned, de-duped, on-target bases out of the PF bases available.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PF_READS">PF_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of reads that pass the vendor's filter.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PF_UNIQUE_READS">PF_UNIQUE_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF reads that are not marked as duplicates.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PF_UQ_BASES_ALIGNED">PF_UQ_BASES_ALIGNED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of bases in the PF aligned reads that are mapped to a reference base.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#PF_UQ_READS_ALIGNED">PF_UQ_READS_ALIGNED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF unique reads that are aligned with mapping score > 0 to the reference genome.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#TARGET_TERRITORY">TARGET_TERRITORY</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The unique number of target bases in the experiment where target is usually exons etc.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#TOTAL_READS">TOTAL_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The total number of reads in the SAM or BAM file examine.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#ZERO_CVG_TARGETS_PCT">ZERO_CVG_TARGETS_PCT</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of targets that did not reach coverage=2 over any base.</TD>
</TR>
</TABLE>
&nbsp;<A NAME="fields_inherited_from_class_picard.metrics.MultilevelMetrics"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Fields inherited from class picard.metrics.<A HREF="../../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../../picard/metrics/MultilevelMetrics.html#LIBRARY">LIBRARY</A>, <A HREF="../../../picard/metrics/MultilevelMetrics.html#READ_GROUP">READ_GROUP</A>, <A HREF="../../../picard/metrics/MultilevelMetrics.html#SAMPLE">SAMPLE</A></CODE></TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../../picard/analysis/directed/HsMetrics.html#HsMetrics()">HsMetrics</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class htsjdk.samtools.metrics.MetricBase</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>equals, hashCode, toString</CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ============ FIELD DETAIL =========== -->

<A NAME="field_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Field Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="BAIT_SET"><!-- --></A><H3>
BAIT_SET</H3>
<PRE>
public java.lang.String <B>BAIT_SET</B></PRE>
<DL>
<DD>The name of the bait set used in the hybrid selection.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="GENOME_SIZE"><!-- --></A><H3>
GENOME_SIZE</H3>
<PRE>
public long <B>GENOME_SIZE</B></PRE>
<DL>
<DD>The number of bases in the reference genome used for alignment.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="BAIT_TERRITORY"><!-- --></A><H3>
BAIT_TERRITORY</H3>
<PRE>
public long <B>BAIT_TERRITORY</B></PRE>
<DL>
<DD>The number of bases which have one or more baits on top of them.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="TARGET_TERRITORY"><!-- --></A><H3>
TARGET_TERRITORY</H3>
<PRE>
public long <B>TARGET_TERRITORY</B></PRE>
<DL>
<DD>The unique number of target bases in the experiment where target is usually exons etc.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="BAIT_DESIGN_EFFICIENCY"><!-- --></A><H3>
BAIT_DESIGN_EFFICIENCY</H3>
<PRE>
public double <B>BAIT_DESIGN_EFFICIENCY</B></PRE>
<DL>
<DD>Target terrirtoy / bait territory.  1 == perfectly efficient, 0.5 = half of baited bases are not target.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="TOTAL_READS"><!-- --></A><H3>
TOTAL_READS</H3>
<PRE>
public long <B>TOTAL_READS</B></PRE>
<DL>
<DD>The total number of reads in the SAM or BAM file examine.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_READS"><!-- --></A><H3>
PF_READS</H3>
<PRE>
public long <B>PF_READS</B></PRE>
<DL>
<DD>The number of reads that pass the vendor's filter.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_UNIQUE_READS"><!-- --></A><H3>
PF_UNIQUE_READS</H3>
<PRE>
public long <B>PF_UNIQUE_READS</B></PRE>
<DL>
<DD>The number of PF reads that are not marked as duplicates.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_PF_READS"><!-- --></A><H3>
PCT_PF_READS</H3>
<PRE>
public double <B>PCT_PF_READS</B></PRE>
<DL>
<DD>PF reads / total reads.  The percent of reads passing filter.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_PF_UQ_READS"><!-- --></A><H3>
PCT_PF_UQ_READS</H3>
<PRE>
public double <B>PCT_PF_UQ_READS</B></PRE>
<DL>
<DD>PF Unique Reads / Total Reads.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_UQ_READS_ALIGNED"><!-- --></A><H3>
PF_UQ_READS_ALIGNED</H3>
<PRE>
public long <B>PF_UQ_READS_ALIGNED</B></PRE>
<DL>
<DD>The number of PF unique reads that are aligned with mapping score > 0 to the reference genome.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_PF_UQ_READS_ALIGNED"><!-- --></A><H3>
PCT_PF_UQ_READS_ALIGNED</H3>
<PRE>
public double <B>PCT_PF_UQ_READS_ALIGNED</B></PRE>
<DL>
<DD>PF Reads Aligned / PF Reads.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_UQ_BASES_ALIGNED"><!-- --></A><H3>
PF_UQ_BASES_ALIGNED</H3>
<PRE>
public long <B>PF_UQ_BASES_ALIGNED</B></PRE>
<DL>
<DD>The number of bases in the PF aligned reads that are mapped to a reference base. Accounts for clipping and gaps.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="ON_BAIT_BASES"><!-- --></A><H3>
ON_BAIT_BASES</H3>
<PRE>
public long <B>ON_BAIT_BASES</B></PRE>
<DL>
<DD>The number of PF aligned bases that mapped to a baited region of the genome.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="NEAR_BAIT_BASES"><!-- --></A><H3>
NEAR_BAIT_BASES</H3>
<PRE>
public long <B>NEAR_BAIT_BASES</B></PRE>
<DL>
<DD>The number of PF aligned bases that mapped to within a fixed interval of a baited region, but not on a baited region.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="OFF_BAIT_BASES"><!-- --></A><H3>
OFF_BAIT_BASES</H3>
<PRE>
public long <B>OFF_BAIT_BASES</B></PRE>
<DL>
<DD>The number of PF aligned bases that mapped to neither on or near a bait.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="ON_TARGET_BASES"><!-- --></A><H3>
ON_TARGET_BASES</H3>
<PRE>
public long <B>ON_TARGET_BASES</B></PRE>
<DL>
<DD>The number of PF aligned bases that mapped to a targeted region of the genome.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_SELECTED_BASES"><!-- --></A><H3>
PCT_SELECTED_BASES</H3>
<PRE>
public double <B>PCT_SELECTED_BASES</B></PRE>
<DL>
<DD>On+Near Bait Bases / PF Bases Aligned.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_OFF_BAIT"><!-- --></A><H3>
PCT_OFF_BAIT</H3>
<PRE>
public double <B>PCT_OFF_BAIT</B></PRE>
<DL>
<DD>The percentage of aligned PF bases that mapped neither on or near a bait.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="ON_BAIT_VS_SELECTED"><!-- --></A><H3>
ON_BAIT_VS_SELECTED</H3>
<PRE>
public double <B>ON_BAIT_VS_SELECTED</B></PRE>
<DL>
<DD>The percentage of on+near bait bases that are on as opposed to near.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEAN_BAIT_COVERAGE"><!-- --></A><H3>
MEAN_BAIT_COVERAGE</H3>
<PRE>
public double <B>MEAN_BAIT_COVERAGE</B></PRE>
<DL>
<DD>The mean coverage of all baits in the experiment.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEAN_TARGET_COVERAGE"><!-- --></A><H3>
MEAN_TARGET_COVERAGE</H3>
<PRE>
public double <B>MEAN_TARGET_COVERAGE</B></PRE>
<DL>
<DD>The mean coverage of targets that received at least coverage depth = 2 at one base.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_USABLE_BASES_ON_BAIT"><!-- --></A><H3>
PCT_USABLE_BASES_ON_BAIT</H3>
<PRE>
public double <B>PCT_USABLE_BASES_ON_BAIT</B></PRE>
<DL>
<DD>The number of aligned, de-duped, on-bait bases out of the PF bases available.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_USABLE_BASES_ON_TARGET"><!-- --></A><H3>
PCT_USABLE_BASES_ON_TARGET</H3>
<PRE>
public double <B>PCT_USABLE_BASES_ON_TARGET</B></PRE>
<DL>
<DD>The number of aligned, de-duped, on-target bases out of the PF bases available.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="FOLD_ENRICHMENT"><!-- --></A><H3>
FOLD_ENRICHMENT</H3>
<PRE>
public double <B>FOLD_ENRICHMENT</B></PRE>
<DL>
<DD>The fold by which the baited region has been amplified above genomic background.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="ZERO_CVG_TARGETS_PCT"><!-- --></A><H3>
ZERO_CVG_TARGETS_PCT</H3>
<PRE>
public double <B>ZERO_CVG_TARGETS_PCT</B></PRE>
<DL>
<DD>The number of targets that did not reach coverage=2 over any base.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="FOLD_80_BASE_PENALTY"><!-- --></A><H3>
FOLD_80_BASE_PENALTY</H3>
<PRE>
public double <B>FOLD_80_BASE_PENALTY</B></PRE>
<DL>
<DD>The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_2X"><!-- --></A><H3>
PCT_TARGET_BASES_2X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_2X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 2X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_10X"><!-- --></A><H3>
PCT_TARGET_BASES_10X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_10X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 10X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_20X"><!-- --></A><H3>
PCT_TARGET_BASES_20X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_20X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 20X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_30X"><!-- --></A><H3>
PCT_TARGET_BASES_30X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_30X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 30X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_40X"><!-- --></A><H3>
PCT_TARGET_BASES_40X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_40X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 40X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_50X"><!-- --></A><H3>
PCT_TARGET_BASES_50X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_50X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 50X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_TARGET_BASES_100X"><!-- --></A><H3>
PCT_TARGET_BASES_100X</H3>
<PRE>
public double <B>PCT_TARGET_BASES_100X</B></PRE>
<DL>
<DD>The percentage of ALL target bases achieving 100X or greater coverage.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_LIBRARY_SIZE"><!-- --></A><H3>
HS_LIBRARY_SIZE</H3>
<PRE>
public java.lang.Long <B>HS_LIBRARY_SIZE</B></PRE>
<DL>
<DD>The estimated number of unique molecules in the selected part of the library.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_PENALTY_10X"><!-- --></A><H3>
HS_PENALTY_10X</H3>
<PRE>
public double <B>HS_PENALTY_10X</B></PRE>
<DL>
<DD>The "hybrid selection penalty" incurred to get 80% of target bases to 10X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 10X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 10 * HS_PENALTY_10X.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_PENALTY_20X"><!-- --></A><H3>
HS_PENALTY_20X</H3>
<PRE>
public double <B>HS_PENALTY_20X</B></PRE>
<DL>
<DD>The "hybrid selection penalty" incurred to get 80% of target bases to 20X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 20X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 20 * HS_PENALTY_20X.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_PENALTY_30X"><!-- --></A><H3>
HS_PENALTY_30X</H3>
<PRE>
public double <B>HS_PENALTY_30X</B></PRE>
<DL>
<DD>The "hybrid selection penalty" incurred to get 80% of target bases to 30X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 30X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 30 * HS_PENALTY_30X.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_PENALTY_40X"><!-- --></A><H3>
HS_PENALTY_40X</H3>
<PRE>
public double <B>HS_PENALTY_40X</B></PRE>
<DL>
<DD>The "hybrid selection penalty" incurred to get 80% of target bases to 40X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 40X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 40 * HS_PENALTY_40X.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_PENALTY_50X"><!-- --></A><H3>
HS_PENALTY_50X</H3>
<PRE>
public double <B>HS_PENALTY_50X</B></PRE>
<DL>
<DD>The "hybrid selection penalty" incurred to get 80% of target bases to 50X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 50X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 50 * HS_PENALTY_50X.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="HS_PENALTY_100X"><!-- --></A><H3>
HS_PENALTY_100X</H3>
<PRE>
public double <B>HS_PENALTY_100X</B></PRE>
<DL>
<DD>The "hybrid selection penalty" incurred to get 80% of target bases to 100X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 100X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 100 * HS_PENALTY_100X.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="AT_DROPOUT"><!-- --></A><H3>
AT_DROPOUT</H3>
<PRE>
public double <B>AT_DROPOUT</B></PRE>
<DL>
<DD>A measure of how undercovered <= 50% GC regions are relative to the mean. For each GC bin [0..50]
 we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
 AT DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
 reads that should have mapped to GC<=50% regions mapped elsewhere.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="GC_DROPOUT"><!-- --></A><H3>
GC_DROPOUT</H3>
<PRE>
public double <B>GC_DROPOUT</B></PRE>
<DL>
<DD>A measure of how undercovered >= 50% GC regions are relative to the mean. For each GC bin [50..100]
 we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
 GC DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
 reads that should have mapped to GC>=50% regions mapped elsewhere.
<P>
<DL>
</DL>
</DL>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="HsMetrics()"><!-- --></A><H3>
HsMetrics</H3>
<PRE>
public <B>HsMetrics</B>()</PRE>
<DL>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/HsMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../../picard/analysis/directed/HsMetricCollector.html" title="class in picard.analysis.directed"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../../picard/analysis/directed/InsertSizeMetricsCollector.html" title="class in picard.analysis.directed"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../../index.html?picard/analysis/directed/HsMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="HsMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
