<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:41 EST 2015 -->
<TITLE>
AbstractAlignmentMerger (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="AbstractAlignmentMerger (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/AbstractAlignmentMerger.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;PREV CLASS&nbsp;
&nbsp;<A HREF="../../picard/sam/AddCommentsToBam.html" title="class in picard.sam"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/sam/AbstractAlignmentMerger.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="AbstractAlignmentMerger.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.sam</FONT>
<BR>
Class AbstractAlignmentMerger</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../resources/inherit.gif" ALT="extended by "><B>picard.sam.AbstractAlignmentMerger</B>
</PRE>
<DL>
<DT><B>Direct Known Subclasses:</B> <DD><A HREF="../../picard/sam/SamAlignmentMerger.html" title="class in picard.sam">SamAlignmentMerger</A></DD>
</DL>
<HR>
<DL>
<DT><PRE>public abstract class <B>AbstractAlignmentMerger</B><DT>extends java.lang.Object</DL>
</PRE>

<P>
Abstract class that coordinates the general task of taking in a set of alignment information,
 possibly in SAM format, possibly in other formats, and merging that with the set of all reads
 for which alignment was attempted, stored in an unmapped SAM file.
 <p/>
 The order of processing is as follows:
 <p/>
 1.  Get records from the unmapped bam and the alignment data
 2.  Merge the alignment information and public tags ONLY from the aligned SAMRecords
 3.  Do additional modifications -- handle clipping, trimming, etc.
 4.  Fix up mate information on paired reads
 5.  Do a final calculation of the NM and UQ tags.
 6.  Write the records to the output file.
 <p/>
 Concrete subclasses which extend AbstractAlignmentMerger should implement getQueryNameSortedAlignedRecords.
 If these records are not in queryname order, mergeAlignment will throw an IllegalStateException.
 <p/>
 Subclasses may optionally implement ignoreAlignment(), which can be used to skip over certain alignments.
<P>

<P>
<DL>
<DT><B>Author:</B></DT>
  <DD>ktibbett@broadinstitute.org</DD>
</DL>
<HR>

<P>
<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;int</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#MAX_RECORDS_IN_RAM">MAX_RECORDS_IN_RAM</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;java.io.File</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#referenceFasta">referenceFasta</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#AbstractAlignmentMerger(java.io.File, java.io.File, java.io.File, boolean, boolean, boolean, htsjdk.samtools.SAMProgramRecord, java.util.List, java.util.List, java.lang.Integer, java.lang.Integer, java.util.List, htsjdk.samtools.SAMFileHeader.SortOrder, picard.sam.PrimaryAlignmentSelectionStrategy, boolean, boolean)">AbstractAlignmentMerger</A></B>(java.io.File&nbsp;unmappedBamFile,
                        java.io.File&nbsp;targetBamFile,
                        java.io.File&nbsp;referenceFasta,
                        boolean&nbsp;clipAdapters,
                        boolean&nbsp;bisulfiteSequence,
                        boolean&nbsp;alignedReadsOnly,
                        htsjdk.samtools.SAMProgramRecord&nbsp;programRecord,
                        java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRetain,
                        java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRemove,
                        java.lang.Integer&nbsp;read1BasesTrimmed,
                        java.lang.Integer&nbsp;read2BasesTrimmed,
                        java.util.List&lt;htsjdk.samtools.SamPairUtil.PairOrientation&gt;&nbsp;expectedOrientations,
                        htsjdk.samtools.SAMFileHeader.SortOrder&nbsp;sortOrder,
                        <A HREF="../../picard/sam/PrimaryAlignmentSelectionStrategy.html" title="interface in picard.sam">PrimaryAlignmentSelectionStrategy</A>&nbsp;primaryAlignmentSelectionStrategy,
                        boolean&nbsp;addMateCigar,
                        boolean&nbsp;unmapContaminantReads)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Constructor</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#clipForOverlappingReads(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)">clipForOverlappingReads</A></B>(htsjdk.samtools.SAMRecord&nbsp;read1,
                        htsjdk.samtools.SAMRecord&nbsp;read2)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Checks to see whether the ends of the reads overlap and soft clips reads
 them if necessary.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#close()">close</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#createNewCigarsIfMapsOffEndOfReference(htsjdk.samtools.SAMRecord)">createNewCigarsIfMapsOffEndOfReference</A></B>(htsjdk.samtools.SAMRecord&nbsp;rec)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Soft-clip an alignment that hangs off the end of its reference sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;htsjdk.samtools.SAMFileHeader</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#getHeader()">getHeader</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;htsjdk.samtools.SAMProgramRecord</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#getProgramRecord()">getProgramRecord</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected abstract &nbsp;htsjdk.samtools.util.CloseableIterator&lt;htsjdk.samtools.SAMRecord&gt;</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#getQuerynameSortedAlignedRecords()">getQuerynameSortedAlignedRecords</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;htsjdk.samtools.SAMSequenceDictionary</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#getSequenceDictionary()">getSequenceDictionary</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#ignoreAlignment(htsjdk.samtools.SAMRecord)">ignoreAlignment</A></B>(htsjdk.samtools.SAMRecord&nbsp;sam)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#isClipOverlappingReads()">isClipOverlappingReads</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#isContaminant(picard.sam.HitsForInsert)">isContaminant</A></B>(picard.sam.HitsForInsert&nbsp;hits)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#isKeepAlignerProperPairFlags()">isKeepAlignerProperPairFlags</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#isReservedTag(java.lang.String)">isReservedTag</A></B>(java.lang.String&nbsp;tag)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#mergeAlignment(java.io.File)">mergeAlignment</A></B>(java.io.File&nbsp;referenceFasta)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Merges the alignment data with the non-aligned records from the source BAM file.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#resetRefSeqFileWalker()">resetRefSeqFileWalker</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#setClipOverlappingReads(boolean)">setClipOverlappingReads</A></B>(boolean&nbsp;clipOverlappingReads)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#setIncludeSecondaryAlignments(boolean)">setIncludeSecondaryAlignments</A></B>(boolean&nbsp;includeSecondaryAlignments)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#setKeepAlignerProperPairFlags(boolean)">setKeepAlignerProperPairFlags</A></B>(boolean&nbsp;keepAlignerProperPairFlags)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;If true, keep the aligner's idea of proper pairs rather than letting alignment merger decide.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#setMaxRecordsInRam(int)">setMaxRecordsInRam</A></B>(int&nbsp;maxRecordsInRam)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Allows the caller to override the maximum records in RAM.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#setProgramRecord(htsjdk.samtools.SAMProgramRecord)">setProgramRecord</A></B>(htsjdk.samtools.SAMProgramRecord&nbsp;pg)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#setValuesFromAlignment(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)">setValuesFromAlignment</A></B>(htsjdk.samtools.SAMRecord&nbsp;rec,
                       htsjdk.samtools.SAMRecord&nbsp;alignment)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sets the values from the alignment record on the unaligned BAM record.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/AbstractAlignmentMerger.html#updateCigarForTrimmedOrClippedBases(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)">updateCigarForTrimmedOrClippedBases</A></B>(htsjdk.samtools.SAMRecord&nbsp;rec,
                                    htsjdk.samtools.SAMRecord&nbsp;alignment)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, equals, finalize, getClass, hashCode, notify, notifyAll, toString, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ============ FIELD DETAIL =========== -->

<A NAME="field_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Field Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="MAX_RECORDS_IN_RAM"><!-- --></A><H3>
MAX_RECORDS_IN_RAM</H3>
<PRE>
public static final int <B>MAX_RECORDS_IN_RAM</B></PRE>
<DL>
<DL>
<DT><B>See Also:</B><DD><A HREF="../../constant-values.html#picard.sam.AbstractAlignmentMerger.MAX_RECORDS_IN_RAM">Constant Field Values</A></DL>
</DL>
<HR>

<A NAME="referenceFasta"><!-- --></A><H3>
referenceFasta</H3>
<PRE>
protected final java.io.File <B>referenceFasta</B></PRE>
<DL>
<DL>
</DL>
</DL>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="AbstractAlignmentMerger(java.io.File, java.io.File, java.io.File, boolean, boolean, boolean, htsjdk.samtools.SAMProgramRecord, java.util.List, java.util.List, java.lang.Integer, java.lang.Integer, java.util.List, htsjdk.samtools.SAMFileHeader.SortOrder, picard.sam.PrimaryAlignmentSelectionStrategy, boolean, boolean)"><!-- --></A><H3>
AbstractAlignmentMerger</H3>
<PRE>
public <B>AbstractAlignmentMerger</B>(java.io.File&nbsp;unmappedBamFile,
                               java.io.File&nbsp;targetBamFile,
                               java.io.File&nbsp;referenceFasta,
                               boolean&nbsp;clipAdapters,
                               boolean&nbsp;bisulfiteSequence,
                               boolean&nbsp;alignedReadsOnly,
                               htsjdk.samtools.SAMProgramRecord&nbsp;programRecord,
                               java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRetain,
                               java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRemove,
                               java.lang.Integer&nbsp;read1BasesTrimmed,
                               java.lang.Integer&nbsp;read2BasesTrimmed,
                               java.util.List&lt;htsjdk.samtools.SamPairUtil.PairOrientation&gt;&nbsp;expectedOrientations,
                               htsjdk.samtools.SAMFileHeader.SortOrder&nbsp;sortOrder,
                               <A HREF="../../picard/sam/PrimaryAlignmentSelectionStrategy.html" title="interface in picard.sam">PrimaryAlignmentSelectionStrategy</A>&nbsp;primaryAlignmentSelectionStrategy,
                               boolean&nbsp;addMateCigar,
                               boolean&nbsp;unmapContaminantReads)</PRE>
<DL>
<DD>Constructor
<P>
<DL>
<DT><B>Parameters:</B><DD><CODE>unmappedBamFile</CODE> - The BAM file that was used as the input to the aligner, which will
                                          include info on all the reads that did not map.  Required.<DD><CODE>targetBamFile</CODE> - The file to which to write the merged SAM records. Required.<DD><CODE>referenceFasta</CODE> - The reference sequence for the map files. Required.<DD><CODE>clipAdapters</CODE> - Whether adapters marked in unmapped BAM file should be marked as
                                          soft clipped in the merged bam. Required.<DD><CODE>bisulfiteSequence</CODE> - Whether the reads are bisulfite sequence (used when calculating the
                                          NM and UQ tags). Required.<DD><CODE>alignedReadsOnly</CODE> - Whether to output only those reads that have alignment data<DD><CODE>programRecord</CODE> - Program record for target file SAMRecords created.<DD><CODE>attributesToRetain</CODE> - private attributes from the alignment record that should be
                                          included when merging.  This overrides the exclusion of
                                          attributes whose tags start with the reserved characters
                                          of X, Y, and Z<DD><CODE>attributesToRemove</CODE> - attributes from the alignment record that should be
                                          removed when merging.  This overrides attributesToRetain if they share
                                          common tags.<DD><CODE>read1BasesTrimmed</CODE> - The number of bases trimmed from start of read 1 prior to alignment.  Optional.<DD><CODE>read2BasesTrimmed</CODE> - The number of bases trimmed from start of read 2 prior to alignment.  Optional.<DD><CODE>expectedOrientations</CODE> - A List of SamPairUtil.PairOrientations that are expected for
                                          aligned pairs.  Used to determine the properPair flag.<DD><CODE>sortOrder</CODE> - The order in which the merged records should be output.  If null,
                                          output will be coordinate-sorted<DD><CODE>primaryAlignmentSelectionStrategy</CODE> - What to do when there are multiple primary alignments, or multiple
                                          alignments but none primary, for a read or read pair.<DD><CODE>addMateCigar</CODE> - True if we are to add or maintain the mate CIGAR (MC) tag, false if we are to remove or not include.<DD><CODE>unmapContaminantReads</CODE> - If true, identify reads having the signature of contamination from a foreign organism (i.e. mostly clipped bases),
                                          and mark them as unmapped.</DL>
</DL>

<!-- ============ METHOD DETAIL ========== -->

<A NAME="method_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Method Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="getQuerynameSortedAlignedRecords()"><!-- --></A><H3>
getQuerynameSortedAlignedRecords</H3>
<PRE>
protected abstract htsjdk.samtools.util.CloseableIterator&lt;htsjdk.samtools.SAMRecord&gt; <B>getQuerynameSortedAlignedRecords</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="ignoreAlignment(htsjdk.samtools.SAMRecord)"><!-- --></A><H3>
ignoreAlignment</H3>
<PRE>
protected boolean <B>ignoreAlignment</B>(htsjdk.samtools.SAMRecord&nbsp;sam)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="isContaminant(picard.sam.HitsForInsert)"><!-- --></A><H3>
isContaminant</H3>
<PRE>
protected boolean <B>isContaminant</B>(picard.sam.HitsForInsert&nbsp;hits)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="setMaxRecordsInRam(int)"><!-- --></A><H3>
setMaxRecordsInRam</H3>
<PRE>
public void <B>setMaxRecordsInRam</B>(int&nbsp;maxRecordsInRam)</PRE>
<DL>
<DD>Allows the caller to override the maximum records in RAM.
<P>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="mergeAlignment(java.io.File)"><!-- --></A><H3>
mergeAlignment</H3>
<PRE>
public void <B>mergeAlignment</B>(java.io.File&nbsp;referenceFasta)</PRE>
<DL>
<DD>Merges the alignment data with the non-aligned records from the source BAM file.
<P>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="clipForOverlappingReads(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)"><!-- --></A><H3>
clipForOverlappingReads</H3>
<PRE>
protected void <B>clipForOverlappingReads</B>(htsjdk.samtools.SAMRecord&nbsp;read1,
                                       htsjdk.samtools.SAMRecord&nbsp;read2)</PRE>
<DL>
<DD>Checks to see whether the ends of the reads overlap and soft clips reads
 them if necessary.
<P>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="setValuesFromAlignment(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)"><!-- --></A><H3>
setValuesFromAlignment</H3>
<PRE>
protected void <B>setValuesFromAlignment</B>(htsjdk.samtools.SAMRecord&nbsp;rec,
                                      htsjdk.samtools.SAMRecord&nbsp;alignment)</PRE>
<DL>
<DD>Sets the values from the alignment record on the unaligned BAM record.  This
 preserves all data from the unaligned record (ReadGroup, NoiseRead status, etc)
 and adds all the alignment info
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>rec</CODE> - The unaligned read record<DD><CODE>alignment</CODE> - The alignment record</DL>
</DD>
</DL>
<HR>

<A NAME="createNewCigarsIfMapsOffEndOfReference(htsjdk.samtools.SAMRecord)"><!-- --></A><H3>
createNewCigarsIfMapsOffEndOfReference</H3>
<PRE>
public static void <B>createNewCigarsIfMapsOffEndOfReference</B>(htsjdk.samtools.SAMRecord&nbsp;rec)</PRE>
<DL>
<DD>Soft-clip an alignment that hangs off the end of its reference sequence.  Checks both the read and its mate,
 if available.
<P>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>rec</CODE> - </DL>
</DD>
</DL>
<HR>

<A NAME="updateCigarForTrimmedOrClippedBases(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)"><!-- --></A><H3>
updateCigarForTrimmedOrClippedBases</H3>
<PRE>
protected void <B>updateCigarForTrimmedOrClippedBases</B>(htsjdk.samtools.SAMRecord&nbsp;rec,
                                                   htsjdk.samtools.SAMRecord&nbsp;alignment)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="getSequenceDictionary()"><!-- --></A><H3>
getSequenceDictionary</H3>
<PRE>
protected htsjdk.samtools.SAMSequenceDictionary <B>getSequenceDictionary</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="getProgramRecord()"><!-- --></A><H3>
getProgramRecord</H3>
<PRE>
protected htsjdk.samtools.SAMProgramRecord <B>getProgramRecord</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="setProgramRecord(htsjdk.samtools.SAMProgramRecord)"><!-- --></A><H3>
setProgramRecord</H3>
<PRE>
protected void <B>setProgramRecord</B>(htsjdk.samtools.SAMProgramRecord&nbsp;pg)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="isReservedTag(java.lang.String)"><!-- --></A><H3>
isReservedTag</H3>
<PRE>
protected boolean <B>isReservedTag</B>(java.lang.String&nbsp;tag)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="getHeader()"><!-- --></A><H3>
getHeader</H3>
<PRE>
protected htsjdk.samtools.SAMFileHeader <B>getHeader</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="resetRefSeqFileWalker()"><!-- --></A><H3>
resetRefSeqFileWalker</H3>
<PRE>
protected void <B>resetRefSeqFileWalker</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="isClipOverlappingReads()"><!-- --></A><H3>
isClipOverlappingReads</H3>
<PRE>
public boolean <B>isClipOverlappingReads</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="setClipOverlappingReads(boolean)"><!-- --></A><H3>
setClipOverlappingReads</H3>
<PRE>
public void <B>setClipOverlappingReads</B>(boolean&nbsp;clipOverlappingReads)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="isKeepAlignerProperPairFlags()"><!-- --></A><H3>
isKeepAlignerProperPairFlags</H3>
<PRE>
public boolean <B>isKeepAlignerProperPairFlags</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="setKeepAlignerProperPairFlags(boolean)"><!-- --></A><H3>
setKeepAlignerProperPairFlags</H3>
<PRE>
public void <B>setKeepAlignerProperPairFlags</B>(boolean&nbsp;keepAlignerProperPairFlags)</PRE>
<DL>
<DD>If true, keep the aligner's idea of proper pairs rather than letting alignment merger decide.
<P>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="setIncludeSecondaryAlignments(boolean)"><!-- --></A><H3>
setIncludeSecondaryAlignments</H3>
<PRE>
public void <B>setIncludeSecondaryAlignments</B>(boolean&nbsp;includeSecondaryAlignments)</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="close()"><!-- --></A><H3>
close</H3>
<PRE>
public void <B>close</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/AbstractAlignmentMerger.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;PREV CLASS&nbsp;
&nbsp;<A HREF="../../picard/sam/AddCommentsToBam.html" title="class in picard.sam"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/sam/AbstractAlignmentMerger.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="AbstractAlignmentMerger.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
