<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:42 EST 2015 -->
<TITLE>
SamAlignmentMerger (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="SamAlignmentMerger (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/SamAlignmentMerger.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/sam/RevertSam.html" title="class in picard.sam"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/sam/SamFormatConverter.html" title="class in picard.sam"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/sam/SamAlignmentMerger.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="SamAlignmentMerger.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#fields_inherited_from_class_picard.sam.AbstractAlignmentMerger">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.sam</FONT>
<BR>
Class SamAlignmentMerger</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../resources/inherit.gif" ALT="extended by "><A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">picard.sam.AbstractAlignmentMerger</A>
      <IMG SRC="../../resources/inherit.gif" ALT="extended by "><B>picard.sam.SamAlignmentMerger</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>SamAlignmentMerger</B><DT>extends <A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></DL>
</PRE>

<P>
Class that takes in a set of alignment information in SAM format and merges it with the set
 of all reads for which alignment was attempted, stored in an unmapped SAM file.  This
 class overrides mergeAlignment in AbstractAlignmentNMerger and proceeds on the assumption that
 the underlying alignment records are aleady in query-name sorted order (true for bwa).  If
 they are not, the mergeAlignment method catches the IllegalStateException, forces a sort
 of the underlying alignment records, and tries again.
<P>

<P>
<DL>
<DT><B>Author:</B></DT>
  <DD>ktibbett@broadinstitute.org</DD>
</DL>
<HR>

<P>
<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
</TABLE>
&nbsp;<A NAME="fields_inherited_from_class_picard.sam.AbstractAlignmentMerger"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Fields inherited from class picard.sam.<A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html#MAX_RECORDS_IN_RAM">MAX_RECORDS_IN_RAM</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#referenceFasta">referenceFasta</A></CODE></TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../picard/sam/SamAlignmentMerger.html#SamAlignmentMerger(java.io.File, java.io.File, java.io.File, htsjdk.samtools.SAMProgramRecord, boolean, boolean, boolean, java.util.List, int, java.util.List, java.util.List, java.lang.Integer, java.lang.Integer, java.util.List, java.util.List, java.util.List, htsjdk.samtools.SAMFileHeader.SortOrder, picard.sam.PrimaryAlignmentSelectionStrategy, boolean, boolean, int)">SamAlignmentMerger</A></B>(java.io.File&nbsp;unmappedBamFile,
                   java.io.File&nbsp;targetBamFile,
                   java.io.File&nbsp;referenceFasta,
                   htsjdk.samtools.SAMProgramRecord&nbsp;programRecord,
                   boolean&nbsp;clipAdapters,
                   boolean&nbsp;bisulfiteSequence,
                   boolean&nbsp;alignedReadsOnly,
                   java.util.List&lt;java.io.File&gt;&nbsp;alignedSamFile,
                   int&nbsp;maxGaps,
                   java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRetain,
                   java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRemove,
                   java.lang.Integer&nbsp;read1BasesTrimmed,
                   java.lang.Integer&nbsp;read2BasesTrimmed,
                   java.util.List&lt;java.io.File&gt;&nbsp;read1AlignedSamFile,
                   java.util.List&lt;java.io.File&gt;&nbsp;read2AlignedSamFile,
                   java.util.List&lt;htsjdk.samtools.SamPairUtil.PairOrientation&gt;&nbsp;expectedOrientations,
                   htsjdk.samtools.SAMFileHeader.SortOrder&nbsp;sortOrder,
                   <A HREF="../../picard/sam/PrimaryAlignmentSelectionStrategy.html" title="interface in picard.sam">PrimaryAlignmentSelectionStrategy</A>&nbsp;primaryAlignmentSelectionStrategy,
                   boolean&nbsp;addMateCigar,
                   boolean&nbsp;unmapContaminantReads,
                   int&nbsp;minUnclippedBases)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Constructor</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/SamAlignmentMerger.html#getForceSort()">getForceSort</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;htsjdk.samtools.util.CloseableIterator&lt;htsjdk.samtools.SAMRecord&gt;</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/SamAlignmentMerger.html#getQuerynameSortedAlignedRecords()">getQuerynameSortedAlignedRecords</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Reads the aligned SAM records into a SortingCollection and returns an iterator over that collection</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/SamAlignmentMerger.html#ignoreAlignment(htsjdk.samtools.SAMRecord)">ignoreAlignment</A></B>(htsjdk.samtools.SAMRecord&nbsp;sam)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For now, we only ignore those alignments that have more than <code>maxGaps</code> insertions
 or deletions.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>protected &nbsp;boolean</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/SamAlignmentMerger.html#isContaminant(picard.sam.HitsForInsert)">isContaminant</A></B>(picard.sam.HitsForInsert&nbsp;hits)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Criteria for contaminant reads:
 1.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;void</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/sam/SamAlignmentMerger.html#mergeAlignment(java.io.File)">mergeAlignment</A></B>(java.io.File&nbsp;referenceFasta)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Merges the alignment from the map file with the non-aligned records from the source BAM file.</TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_picard.sam.AbstractAlignmentMerger"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class picard.sam.<A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html#clipForOverlappingReads(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)">clipForOverlappingReads</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#close()">close</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#createNewCigarsIfMapsOffEndOfReference(htsjdk.samtools.SAMRecord)">createNewCigarsIfMapsOffEndOfReference</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#getHeader()">getHeader</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#getProgramRecord()">getProgramRecord</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#getSequenceDictionary()">getSequenceDictionary</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#isClipOverlappingReads()">isClipOverlappingReads</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#isKeepAlignerProperPairFlags()">isKeepAlignerProperPairFlags</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#isReservedTag(java.lang.String)">isReservedTag</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#resetRefSeqFileWalker()">resetRefSeqFileWalker</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#setClipOverlappingReads(boolean)">setClipOverlappingReads</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#setIncludeSecondaryAlignments(boolean)">setIncludeSecondaryAlignments</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#setKeepAlignerProperPairFlags(boolean)">setKeepAlignerProperPairFlags</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#setMaxRecordsInRam(int)">setMaxRecordsInRam</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#setProgramRecord(htsjdk.samtools.SAMProgramRecord)">setProgramRecord</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#setValuesFromAlignment(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)">setValuesFromAlignment</A>, <A HREF="../../picard/sam/AbstractAlignmentMerger.html#updateCigarForTrimmedOrClippedBases(htsjdk.samtools.SAMRecord, htsjdk.samtools.SAMRecord)">updateCigarForTrimmedOrClippedBases</A></CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, equals, finalize, getClass, hashCode, notify, notifyAll, toString, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="SamAlignmentMerger(java.io.File, java.io.File, java.io.File, htsjdk.samtools.SAMProgramRecord, boolean, boolean, boolean, java.util.List, int, java.util.List, java.util.List, java.lang.Integer, java.lang.Integer, java.util.List, java.util.List, java.util.List, htsjdk.samtools.SAMFileHeader.SortOrder, picard.sam.PrimaryAlignmentSelectionStrategy, boolean, boolean, int)"><!-- --></A><H3>
SamAlignmentMerger</H3>
<PRE>
public <B>SamAlignmentMerger</B>(java.io.File&nbsp;unmappedBamFile,
                          java.io.File&nbsp;targetBamFile,
                          java.io.File&nbsp;referenceFasta,
                          htsjdk.samtools.SAMProgramRecord&nbsp;programRecord,
                          boolean&nbsp;clipAdapters,
                          boolean&nbsp;bisulfiteSequence,
                          boolean&nbsp;alignedReadsOnly,
                          java.util.List&lt;java.io.File&gt;&nbsp;alignedSamFile,
                          int&nbsp;maxGaps,
                          java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRetain,
                          java.util.List&lt;java.lang.String&gt;&nbsp;attributesToRemove,
                          java.lang.Integer&nbsp;read1BasesTrimmed,
                          java.lang.Integer&nbsp;read2BasesTrimmed,
                          java.util.List&lt;java.io.File&gt;&nbsp;read1AlignedSamFile,
                          java.util.List&lt;java.io.File&gt;&nbsp;read2AlignedSamFile,
                          java.util.List&lt;htsjdk.samtools.SamPairUtil.PairOrientation&gt;&nbsp;expectedOrientations,
                          htsjdk.samtools.SAMFileHeader.SortOrder&nbsp;sortOrder,
                          <A HREF="../../picard/sam/PrimaryAlignmentSelectionStrategy.html" title="interface in picard.sam">PrimaryAlignmentSelectionStrategy</A>&nbsp;primaryAlignmentSelectionStrategy,
                          boolean&nbsp;addMateCigar,
                          boolean&nbsp;unmapContaminantReads,
                          int&nbsp;minUnclippedBases)</PRE>
<DL>
<DD>Constructor
<P>
<DL>
<DT><B>Parameters:</B><DD><CODE>unmappedBamFile</CODE> - The BAM file that was used as the input to the aligner, which will
                                          include info on all the reads that did not map.  Required.<DD><CODE>targetBamFile</CODE> - The file to which to write the merged SAM records. Required.<DD><CODE>referenceFasta</CODE> - The reference sequence for the map files. Required.<DD><CODE>programRecord</CODE> - Program record for taget file SAMRecords created.<DD><CODE>clipAdapters</CODE> - Whether adapters marked in unmapped BAM file should be marked as
                                          soft clipped in the merged bam. Required.<DD><CODE>bisulfiteSequence</CODE> - Whether the reads are bisulfite sequence (used when calculating the
                                          NM and UQ tags). Required.<DD><CODE>alignedReadsOnly</CODE> - Whether to output only those reads that have alignment data<DD><CODE>alignedSamFile</CODE> - The SAM file(s) with alignment information.  Optional.  If this is
                                          not provided, then read1AlignedSamFile and read2AlignedSamFile must be.<DD><CODE>maxGaps</CODE> - The maximum number of insertions or deletions permitted in an
                                          alignment.  Alignments with more than this many gaps will be ignored.
                                          -1 means to allow any number of gaps.<DD><CODE>attributesToRetain</CODE> - attributes from the alignment record that should be
                                          removed when merging.  This overrides attributesToRetain if they share
                                          common tags.<DD><CODE>read1BasesTrimmed</CODE> - The number of bases trimmed from start of read 1 prior to alignment.  Optional.<DD><CODE>read2BasesTrimmed</CODE> - The number of bases trimmed from start of read 2 prior to alignment.  Optional.<DD><CODE>read1AlignedSamFile</CODE> - The alignment records for read1.  Used when the two ends of a read are
                                          aligned separately.  This is optional, but must be specified if
                                          alignedSamFile is not.<DD><CODE>read2AlignedSamFile</CODE> - The alignment records for read1.  Used when the two ends of a read are
                                          aligned separately.  This is optional, but must be specified if
                                          alignedSamFile is not.<DD><CODE>expectedOrientations</CODE> - A List of SamPairUtil.PairOrientations that are expected for
                                          aligned pairs.  Used to determine the properPair flag.<DD><CODE>sortOrder</CODE> - The order in which the merged records should be output.  If null,
                                          output will be coordinate-sorted<DD><CODE>primaryAlignmentSelectionStrategy</CODE> - How to handle multiple alignments for a fragment or read pair,
                                          in which none are primary, or more than one is marked primary<DD><CODE>addMateCigar</CODE> - True if we are to add or maintain the mate CIGAR (MC) tag, false if we are to remove or not include.<DD><CODE>unmapContaminantReads</CODE> - If true, identify reads having the signature of contamination from a foreign organism (i.e. mostly clipped bases),
                                          and mark them as unmapped.<DD><CODE>minUnclippedBases</CODE> - If unmapContaminantReads is set, require this many unclipped bases or else the read will be marked as contaminant.</DL>
</DL>

<!-- ============ METHOD DETAIL ========== -->

<A NAME="method_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Method Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="mergeAlignment(java.io.File)"><!-- --></A><H3>
mergeAlignment</H3>
<PRE>
public void <B>mergeAlignment</B>(java.io.File&nbsp;referenceFasta)</PRE>
<DL>
<DD>Merges the alignment from the map file with the non-aligned records from the source BAM file.
 Overrides mergeAlignment in AbstractAlignmentMerger.  Tries first to proceed on the assumption
 that the alignment records are pre-sorted.  If not, catches the exception, forces a sort, and
 tries again.
<P>
<DD><DL>
<DT><B>Overrides:</B><DD><CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html#mergeAlignment(java.io.File)">mergeAlignment</A></CODE> in class <CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></CODE></DL>
</DD>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="getQuerynameSortedAlignedRecords()"><!-- --></A><H3>
getQuerynameSortedAlignedRecords</H3>
<PRE>
protected htsjdk.samtools.util.CloseableIterator&lt;htsjdk.samtools.SAMRecord&gt; <B>getQuerynameSortedAlignedRecords</B>()</PRE>
<DL>
<DD>Reads the aligned SAM records into a SortingCollection and returns an iterator over that collection
<P>
<DD><DL>
<DT><B>Specified by:</B><DD><CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html#getQuerynameSortedAlignedRecords()">getQuerynameSortedAlignedRecords</A></CODE> in class <CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></CODE></DL>
</DD>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="ignoreAlignment(htsjdk.samtools.SAMRecord)"><!-- --></A><H3>
ignoreAlignment</H3>
<PRE>
protected boolean <B>ignoreAlignment</B>(htsjdk.samtools.SAMRecord&nbsp;sam)</PRE>
<DL>
<DD>For now, we only ignore those alignments that have more than <code>maxGaps</code> insertions
 or deletions.
<P>
<DD><DL>
<DT><B>Overrides:</B><DD><CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html#ignoreAlignment(htsjdk.samtools.SAMRecord)">ignoreAlignment</A></CODE> in class <CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></CODE></DL>
</DD>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="isContaminant(picard.sam.HitsForInsert)"><!-- --></A><H3>
isContaminant</H3>
<PRE>
protected boolean <B>isContaminant</B>(picard.sam.HitsForInsert&nbsp;hits)</PRE>
<DL>
<DD>Criteria for contaminant reads:
 1. primary alignment has fewer than minUnclippedBases unclipped bases
 2. primary alignment has both ends clipped
 3. for pairs, at least one end of primary alignment meets above criteria
<P>
<DD><DL>
<DT><B>Overrides:</B><DD><CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html#isContaminant(picard.sam.HitsForInsert)">isContaminant</A></CODE> in class <CODE><A HREF="../../picard/sam/AbstractAlignmentMerger.html" title="class in picard.sam">AbstractAlignmentMerger</A></CODE></DL>
</DD>
<DD><DL>
</DL>
</DD>
</DL>
<HR>

<A NAME="getForceSort()"><!-- --></A><H3>
getForceSort</H3>
<PRE>
public boolean <B>getForceSort</B>()</PRE>
<DL>
<DD><DL>
</DL>
</DD>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/SamAlignmentMerger.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/sam/RevertSam.html" title="class in picard.sam"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/sam/SamFormatConverter.html" title="class in picard.sam"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/sam/SamAlignmentMerger.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="SamAlignmentMerger.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#fields_inherited_from_class_picard.sam.AbstractAlignmentMerger">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
