<a id="EstimateLibraryComplexity"/>
<h3>EstimateLibraryComplexity</h3>
<section>
<p>Attempts to estimate library complexity from sequence of read pairs alone. Does so by sorting all reads by the first N bases (5 by default) of each read and then comparing reads with the first N bases identical to each other for duplicates.  Reads are considered to be duplicates if they match each other with no gaps and an overall mismatch rate less than or equal to MAX_DIFF_RATE (0.03 by default).

Reads of poor quality are filtered out so as to provide a more accurate estimate. The filtering removes reads with any no-calls in the first N bases or with a mean base quality lower than MIN_MEAN_QUALITY across either the first or second read.

Unpaired reads are ignored in this computation.

The algorithm attempts to detect optical duplicates separately from PCR duplicates and excludes these in the calculation of library size. Also, since there is no alignment to screen out technical reads one further filter is applied on the data.  After examining all reads a Histogram is built of [#reads in duplicate set -> #of duplicate sets] all bins that contain exactly one duplicate set are then removed from the Histogram as outliers before library size is estimated.</p>
<table>
<tr><th>Option</th><th>Description</th></tr>
<tr><td>INPUT (File)</td><td>One or more files to combine and estimate library complexity from. Reads can be mapped or unmapped.  Default value: null. This option may be specified 0 or more times. </td></tr>
<tr><td>OUTPUT (File)</td><td>Output file to writes per-library metrics to.  Required. </td></tr>
<tr><td>MIN_IDENTICAL_BASES (Integer)</td><td>The minimum number of bases at the starts of reads that must be identical for reads to be grouped together for duplicate detection.  In effect total_reads / 4^max_id_bases reads will be compared at a time, so lower numbers will produce more accurate results but consume exponentially more memory and CPU.  Default value: 5. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>MAX_DIFF_RATE (Double)</td><td>The maximum rate of differences between two reads to call them identical.  Default value: 0.03. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>MIN_MEAN_QUALITY (Integer)</td><td>The minimum mean quality of the bases in a read pair for the read to be analyzed. Reads with lower average quality are filtered out and not considered in any calculations.  Default value: 20. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>MAX_GROUP_RATIO (Integer)</td><td>Do not process self-similar groups that are this many times over the mean expected group size. I.e. if the input contains 10m read pairs and MIN_IDENTICAL_BASES is set to 5, then the mean expected group size would be approximately 10 reads.  Default value: 500. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>BARCODE_TAG (String)</td><td>Barcode SAM tag (ex. BC for 10X Genomics)  Default value: null. </td></tr>
<tr><td>READ_ONE_BARCODE_TAG (String)</td><td>Read one barcode SAM tag (ex. BX for 10X Genomics)  Default value: null. </td></tr>
<tr><td>READ_TWO_BARCODE_TAG (String)</td><td>Read two barcode SAM tag (ex. BX for 10X Genomics)  Default value: null. </td></tr>
<tr><td>READ_NAME_REGEX (String)</td><td>Regular expression that can be used to parse read names in the incoming SAM file. Read names are parsed to extract three variables: tile/region, x coordinate and y coordinate. These values are used to estimate the rate of optical duplication in order to give a more accurate estimated library size. Set this option to null to disable optical duplicate detection. The regular expression should contain three capture groups for the three variables, in order. It must match the entire read name. Note that if the default regex is specified, a regex match is not actually done, but instead the read name  is split on colon character. For 5 element names, the 3rd, 4th and 5th elements are assumed to be tile, x and y values. For 7 element names (CASAVA 1.8), the 5th, 6th, and 7th elements are assumed to be tile, x and y values.  Default value: [a-zA-Z0-9]+:[0-9]:([0-9]+):([0-9]+):([0-9]+).*. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>OPTICAL_DUPLICATE_PIXEL_DISTANCE (Integer)</td><td>The maximum offset between two duplicte clusters in order to consider them optical duplicates. This should usually be set to some fairly small number (e.g. 5-10 pixels) unless using later versions of the Illumina pipeline that multiply pixel values by 10, in which case 50-100 is more normal.  Default value: 100. This option can be set to 'null' to clear the default value. </td></tr>
</table>
</section>
