<a id="FastqToSam"/>
<h3>FastqToSam</h3>
<section>
<p>Extracts read sequences and qualities from the input fastq file and writes them into the output file in unaligned BAM format. Input files can be in GZip format (end in .gz).
</p>
<table>
<tr><th>Option</th><th>Description</th></tr>
<tr><td>FASTQ (File)</td><td>Input fastq file (optionally gzipped) for single end data, or first read in paired end data.  Required. </td></tr>
<tr><td>FASTQ2 (File)</td><td>Input fastq file (optionally gzipped) for the second read of paired end data.  Default value: null. </td></tr>
<tr><td>USE_SEQUENTIAL_FASTQS (Boolean)</td><td>Use sequential fastq files with the suffix &lt;prefix>_###.fastq or &lt;prefix>_###.fastq.gz  Default value: false. This option can be set to 'null' to clear the default value. Possible values: {true, false} </td></tr>
<tr><td>QUALITY_FORMAT (FastqQualityFormat)</td><td>A value describing how the quality values are encoded in the fastq.  Either Solexa for pre-pipeline 1.3 style scores (solexa scaling + 66), Illumina for pipeline 1.3 and above (phred scaling + 64) or Standard for phred scaled scores with a character shift of 33.  If this value is not specified, the quality format will be detected automatically.  Default value: null. Possible values: {Solexa, Illumina, Standard} </td></tr>
<tr><td>OUTPUT (File)</td><td>Output SAM/BAM file.   Required. </td></tr>
<tr><td>READ_GROUP_NAME (String)</td><td>Read group name  Default value: A. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>SAMPLE_NAME (String)</td><td>Sample name to insert into the read group header  Required. </td></tr>
<tr><td>LIBRARY_NAME (String)</td><td>The library name to place into the LB attribute in the read group header  Default value: null. </td></tr>
<tr><td>PLATFORM_UNIT (String)</td><td>The platform unit (often run_barcode.lane) to insert into the read group header  Default value: null. </td></tr>
<tr><td>PLATFORM (String)</td><td>The platform type (e.g. illumina, solid) to insert into the read group header  Default value: null. </td></tr>
<tr><td>SEQUENCING_CENTER (String)</td><td>The sequencing center from which the data originated  Default value: null. </td></tr>
<tr><td>PREDICTED_INSERT_SIZE (Integer)</td><td>Predicted median insert size, to insert into the read group header  Default value: null. </td></tr>
<tr><td>PROGRAM_GROUP (String)</td><td>Program group to insert into the read group header.  Default value: null. </td></tr>
<tr><td>PLATFORM_MODEL (String)</td><td>Platform model to insert into the group header (free-form text providing further details of the platform/technology used)  Default value: null. </td></tr>
<tr><td>COMMENT (String)</td><td>Comment(s) to include in the merged output file's header.  Default value: null. This option may be specified 0 or more times. </td></tr>
<tr><td>DESCRIPTION (String)</td><td>Inserted into the read group header  Default value: null. </td></tr>
<tr><td>RUN_DATE (Iso8601Date)</td><td>Date the run was produced, to insert into the read group header  Default value: null. </td></tr>
<tr><td>SORT_ORDER (SortOrder)</td><td>The sort order for the output sam/bam file.  Default value: queryname. This option can be set to 'null' to clear the default value. Possible values: {unsorted, queryname, coordinate, duplicate} </td></tr>
<tr><td>MIN_Q (Integer)</td><td>Minimum quality allowed in the input fastq.  An exception will be thrown if a quality is less than this value.  Default value: 0. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>MAX_Q (Integer)</td><td>Maximum quality allowed in the input fastq.  An exception will be thrown if a quality is greater than this value.  Default value: 93. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>STRIP_UNPAIRED_MATE_NUMBER (Boolean)</td><td>If true and this is an unpaired fastq any occurance of '/1' will be removed from the end of a read name.  Default value: false. This option can be set to 'null' to clear the default value. Possible values: {true, false} </td></tr>
<tr><td>ALLOW_AND_IGNORE_EMPTY_LINES (Boolean)</td><td>Allow (and ignore) empty lines  Default value: false. This option can be set to 'null' to clear the default value. Possible values: {true, false} </td></tr>
</table>
</section>
