<h2>Picard Metrics Definitions</h2>
<section>
<p> Click on a metric to see a description of its fields.</p>
<ol>
<li><a href="#AlignmentSummaryMetrics">AlignmentSummaryMetrics</a>: High level metrics about the alignment of reads within a SAM file, produced by
 the CollectAlignmentSummaryMetrics program and usually stored in a file with
 the extension ".alignment_summary_metrics".</li>
<li><a href="#BaseDistributionByCycleMetrics">BaseDistributionByCycleMetrics</a>: </li>
<li><a href="#CollectHiSeqXPfFailMetrics.PFFailDetailedMetric">CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</a>: a metric class for describing FP failing reads from an Illumina HiSeqX lane *</li>
<li><a href="#CollectHiSeqXPfFailMetrics.PFFailSummaryMetric">CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</a>: Metrics produced by the GetHiSeqXPFFailMetrics program.</li>
<li><a href="#CollectOxoGMetrics.CpcgMetrics">CollectOxoGMetrics.CpcgMetrics</a>: Metrics class for outputs.</li>
<li><a href="#CollectQualityYieldMetrics.QualityYieldMetrics">CollectQualityYieldMetrics.QualityYieldMetrics</a>: A set of metrics used to describe the general quality of a BAM file</li>
<li><a href="#CollectRawWgsMetrics.RawWgsMetrics">CollectRawWgsMetrics.RawWgsMetrics</a>: </li>
<li><a href="#CollectVariantCallingMetrics.VariantCallingDetailMetrics">CollectVariantCallingMetrics.VariantCallingDetailMetrics</a>: A collection of metrics relating to snps and indels within a variant-calling file (VCF) for a given sample.</li>
<li><a href="#CollectVariantCallingMetrics.VariantCallingSummaryMetrics">CollectVariantCallingMetrics.VariantCallingSummaryMetrics</a>: A collection of metrics relating to snps and indels within a variant-calling file (VCF).</li>
<li><a href="#CollectWgsMetrics.WgsMetrics">CollectWgsMetrics.WgsMetrics</a>: Metrics for evaluating the performance of whole genome sequencing experiments.</li>
<li><a href="#CollectWgsMetricsFromQuerySorted.QuerySortedSeqMetrics">CollectWgsMetricsFromQuerySorted.QuerySortedSeqMetrics</a>: Metrics for evaluating the performance of whole genome sequencing experiments.</li>
<li><a href="#CollectWgsMetricsFromSampledSites.SampledWgsMetrics">CollectWgsMetricsFromSampledSites.SampledWgsMetrics</a>: </li>
<li><a href="#DuplicationMetrics">DuplicationMetrics</a>: Metrics that are calculated during the process of marking duplicates
 within a stream of SAMRecords.</li>
<li><a href="#ExtractIlluminaBarcodes.BarcodeMetric">ExtractIlluminaBarcodes.BarcodeMetric</a>: Metrics produced by the ExtractIlluminaBarcodes program that is used to parse data in
 the basecalls directory and determine to which barcode each read should be assigned.</li>
<li><a href="#GcBiasDetailMetrics">GcBiasDetailMetrics</a>: Class that holds detailed metrics about reads that fall within windows of a certain
 GC bin on the reference genome.</li>
<li><a href="#GcBiasMetrics">GcBiasMetrics</a>: </li>
<li><a href="#GcBiasSummaryMetrics">GcBiasSummaryMetrics</a>: High level metrics that capture how biased the coverage in a certain lane is.</li>
<li><a href="#GenotypeConcordanceContingencyMetrics">GenotypeConcordanceContingencyMetrics</a>: Class that holds metrics about the Genotype Concordance contingency tables.</li>
<li><a href="#GenotypeConcordanceDetailMetrics">GenotypeConcordanceDetailMetrics</a>: Class that holds detail metrics about Genotype Concordance</li>
<li><a href="#GenotypeConcordanceSummaryMetrics">GenotypeConcordanceSummaryMetrics</a>: Class that holds summary metrics about Genotype Concordance</li>
<li><a href="#HsMetrics">HsMetrics</a>: The set of metrics captured that are specific to a hybrid selection analysis.</li>
<li><a href="#IlluminaBasecallingMetrics">IlluminaBasecallingMetrics</a>: Metric for Illumina Basecalling that stores means and standard deviations on a per-barcode per-lane basis.</li>
<li><a href="#IlluminaLaneMetrics">IlluminaLaneMetrics</a>: Embodies characteristics that describe a lane.</li>
<li><a href="#IlluminaPhasingMetrics">IlluminaPhasingMetrics</a>: Metrics for Illumina Basecalling that stores median phasing and prephasing percentages on a per-template-read, per-lane basis.</li>
<li><a href="#InsertSizeMetrics">InsertSizeMetrics</a>: Metrics about the insert size distribution of a paired-end library, created by the
 CollectInsertSizeMetrics program and usually written to a file with the extension
 ".insert_size_metrics".</li>
<li><a href="#JumpingLibraryMetrics">JumpingLibraryMetrics</a>: High level metrics about the presence of  outward- and inward-facing pairs
 within a SAM file generated with a jumping library, produced by
 the CollectJumpingLibraryMetrics program and usually stored in a file with
 the extension ".jump_metrics".</li>
<li><a href="#MultilevelMetrics">MultilevelMetrics</a>: </li>
<li><a href="#RnaSeqMetrics">RnaSeqMetrics</a>: Metrics about the alignment of RNA-seq reads within a SAM file to genes, produced by the CollectRnaSeqMetrics
 program and usually stored in a file with the extension ".rna_metrics".</li>
<li><a href="#RrbsCpgDetailMetrics">RrbsCpgDetailMetrics</a>: Holds information about CpG sites encountered for RRBS processing QC</li>
<li><a href="#RrbsSummaryMetrics">RrbsSummaryMetrics</a>: Holds summary statistics from RRBS processing QC</li>
<li><a href="#SamFileValidator.ValidationMetrics">SamFileValidator.ValidationMetrics</a>: </li>
<li><a href="#SequencingArtifactMetrics.BaitBiasDetailMetrics">SequencingArtifactMetrics.BaitBiasDetailMetrics</a>: Bait bias artifacts broken down by context.</li>
<li><a href="#SequencingArtifactMetrics.BaitBiasSummaryMetrics">SequencingArtifactMetrics.BaitBiasSummaryMetrics</a>: Summary analysis of a single bait bias artifact, also known as a reference bias artifact.</li>
<li><a href="#SequencingArtifactMetrics.PreAdapterDetailMetrics">SequencingArtifactMetrics.PreAdapterDetailMetrics</a>: Pre-adapter artifacts broken down by context.</li>
<li><a href="#SequencingArtifactMetrics.PreAdapterSummaryMetrics">SequencingArtifactMetrics.PreAdapterSummaryMetrics</a>: Summary analysis of a single pre-adapter artifact.</li>
<li><a href="#TargetedPcrMetrics">TargetedPcrMetrics</a>: Metrics class for targeted pcr runs such as TSCA runs</li>
</ol>
</section>
<a id="AlignmentSummaryMetrics"></a>
<h2>AlignmentSummaryMetrics</h2>
<section>
<p>High level metrics about the alignment of reads within a SAM file, produced by
 the CollectAlignmentSummaryMetrics program and usually stored in a file with
 the extension ".alignment_summary_metrics".</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>CATEGORY</td><td>One of either UNPAIRED (for a fragment run), FIRST_OF_PAIR when metrics are for only the
 first read in a paired run, SECOND_OF_PAIR when the metrics are for only the second read
 in a paired run or PAIR when the metrics are aggregated for both first and second reads
 in a pair.</td></tr><tr><td>TOTAL_READS</td><td>The total number of reads including all PF and non-PF reads. When CATEGORY equals PAIR
 this value will be 2x the number of clusters.</td></tr><tr><td>PF_READS</td><td>The number of PF reads where PF is defined as passing Illumina's filter.</td></tr><tr><td>PCT_PF_READS</td><td>The percentage of reads that are PF (PF_READS / TOTAL_READS)</td></tr><tr><td>PF_NOISE_READS</td><td>The number of PF reads that are marked as noise reads.  A noise read is one which is composed
 entirely of A bases and/or N bases. These reads are marked as they are usually artifactual and
 are of no use in downstream analysis.</td></tr><tr><td>PF_READS_ALIGNED</td><td>The number of PF reads that were aligned to the reference sequence. This includes reads that
 aligned with low quality (i.e. their alignments are ambiguous).</td></tr><tr><td>PCT_PF_READS_ALIGNED</td><td>The percentage of PF reads that aligned to the reference sequence. PF_READS_ALIGNED / PF_READS</td></tr><tr><td>PF_ALIGNED_BASES</td><td>The total number of aligned bases, in all mapped PF reads, that are aligned to the reference sequence.</td></tr><tr><td>PF_HQ_ALIGNED_READS</td><td>The number of PF reads that were aligned to the reference sequence with a mapping quality of
 Q20 or higher signifying that the aligner estimates a 1/100 (or smaller) chance that the
 alignment is wrong.</td></tr><tr><td>PF_HQ_ALIGNED_BASES</td><td>The number of bases aligned to the reference sequence in reads that were mapped at high
 quality.  Will usually approximate PF_HQ_ALIGNED_READS * READ_LENGTH but may differ when
 either mixed read lengths are present or many reads are aligned with gaps.</td></tr><tr><td>PF_HQ_ALIGNED_Q20_BASES</td><td>The subset of PF_HQ_ALIGNED_BASES where the base call quality was Q20 or higher.</td></tr><tr><td>PF_HQ_MEDIAN_MISMATCHES</td><td>The median number of mismatches versus the reference sequence in reads that were aligned
 to the reference at high quality (i.e. PF_HQ_ALIGNED READS).</td></tr><tr><td>PF_MISMATCH_RATE</td><td>The rate of bases mismatching the reference for all bases aligned to the reference sequence.</td></tr><tr><td>PF_HQ_ERROR_RATE</td><td>The percentage of bases that mismatch the reference in PF HQ aligned reads.</td></tr><tr><td>PF_INDEL_RATE</td><td>The number of insertion and deletion events per 100 aligned bases.  Uses the number of events
 as the numerator, not the number of inserted or deleted bases.</td></tr><tr><td>MEAN_READ_LENGTH</td><td>The mean read length of the set of reads examined.  When looking at the data for a single lane with
 equal length reads this number is just the read length.  When looking at data for merged lanes with
 differing read lengths this is the mean read length of all reads.</td></tr><tr><td>READS_ALIGNED_IN_PAIRS</td><td>The number of aligned reads whose mate pair was also aligned to the reference.</td></tr><tr><td>PCT_READS_ALIGNED_IN_PAIRS</td><td>The percentage of reads whose mate pair was also aligned to the reference.
 READS_ALIGNED_IN_PAIRS / PF_READS_ALIGNED</td></tr><tr><td>BAD_CYCLES</td><td>The number of instrument cycles in which 80% or more of base calls were no-calls.</td></tr><tr><td>STRAND_BALANCE</td><td>The number of PF reads aligned to the positive strand of the genome divided by the number of
 PF reads aligned to the genome.</td></tr><tr><td>PCT_CHIMERAS</td><td>The percentage of reads that map outside of a maximum insert size (usually 100kb) or that have
 the two ends mapping to different chromosomes.</td></tr><tr><td>PCT_ADAPTER</td><td>The percentage of PF reads that are unaligned and match to a known adapter sequence right from the
 start of the read.</td></tr></table>
</section>
<a id="BaseDistributionByCycleMetrics"></a>
<h2>BaseDistributionByCycleMetrics</h2>
<section>
<p></p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>READ_END</td><td></td></tr><tr><td>CYCLE</td><td></td></tr><tr><td>PCT_A</td><td></td></tr><tr><td>PCT_C</td><td></td></tr><tr><td>PCT_G</td><td></td></tr><tr><td>PCT_T</td><td></td></tr><tr><td>PCT_N</td><td></td></tr></table>
</section>
<a id="CollectHiSeqXPfFailMetrics.PFFailDetailedMetric"></a>
<h2>CollectHiSeqXPfFailMetrics.PFFailDetailedMetric</h2>
<section>
<p>a metric class for describing FP failing reads from an Illumina HiSeqX lane *</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>TILE</td><td></td></tr><tr><td>X</td><td></td></tr><tr><td>Y</td><td></td></tr><tr><td>NUM_N</td><td></td></tr><tr><td>NUM_Q_GT_TWO</td><td></td></tr><tr><td>CLASSIFICATION</td><td>The classification of this read: {EMPTY, POLYCLONAL, MISALIGNED, UNKNOWN}
 (See PFFailSummaryMetric for explanation regarding the possible classification.)</td></tr></table>
</section>
<a id="CollectHiSeqXPfFailMetrics.PFFailSummaryMetric"></a>
<h2>CollectHiSeqXPfFailMetrics.PFFailSummaryMetric</h2>
<section>
<p>Metrics produced by the GetHiSeqXPFFailMetrics program. Used to diagnose lanes from HiSeqX
 Sequencing, providing the number and fraction of each of the reasons that reads could have not passed PF.
 Possible reasons are EMPTY (reads from empty wells with no template strand), POLYCLONAL (reads from wells that had more than one strand
 cloned in them), MISALIGNED (reads from wells that are near the edge of the tile), UNKNOWN (reads that didn't pass PF but couldn't be diagnosed)</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>TILE</td><td>The Tile that is described by this metric. Can be a string (like "All") to mean some marginal over tiles. *</td></tr><tr><td>READS</td><td>The total number of reads examined</td></tr><tr><td>PF_FAIL_READS</td><td>The number of non-PF reads in this tile.</td></tr><tr><td>PCT_PF_FAIL_READS</td><td>The fraction of PF_READS</td></tr><tr><td>PF_FAIL_EMPTY</td><td>The number of non-PF reads in this tile that are deemed empty.</td></tr><tr><td>PCT_PF_FAIL_EMPTY</td><td>The fraction of non-PF reads in this tile that are deemed empty (as fraction of all non-PF reads).</td></tr><tr><td>PF_FAIL_POLYCLONAL</td><td>The number of non-PF reads in this tile that are deemed multiclonal.</td></tr><tr><td>PCT_PF_FAIL_POLYCLONAL</td><td>The fraction of non-PF reads in this tile that are deemed multiclonal (as fraction of all non-PF reads).</td></tr><tr><td>PF_FAIL_MISALIGNED</td><td>The number of non-PF reads in this tile that are deemed "misaligned".</td></tr><tr><td>PCT_PF_FAIL_MISALIGNED</td><td>The fraction of non-PF reads in this tile that are deemed "misaligned" (as fraction of all non-PF reads).</td></tr><tr><td>PF_FAIL_UNKNOWN</td><td>The number of non-PF reads in this tile that have not been classified.</td></tr><tr><td>PCT_PF_FAIL_UNKNOWN</td><td>The fraction of non-PF reads in this tile that have not been classified (as fraction of all non-PF reads).</td></tr></table>
</section>
<a id="CollectOxoGMetrics.CpcgMetrics"></a>
<h2>CollectOxoGMetrics.CpcgMetrics</h2>
<section>
<p>Metrics class for outputs.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE_ALIAS</td><td>The name of the sample being assayed.</td></tr><tr><td>LIBRARY</td><td>The name of the library being assayed.</td></tr><tr><td>CONTEXT</td><td>The sequence context being reported on.</td></tr><tr><td>TOTAL_SITES</td><td>The total number of sites that had at least one base covering them.</td></tr><tr><td>TOTAL_BASES</td><td>The total number of basecalls observed at all sites.</td></tr><tr><td>REF_NONOXO_BASES</td><td>The number of reference alleles observed as C in read 1 and G in read 2.</td></tr><tr><td>REF_OXO_BASES</td><td>The number of reference alleles observed as G in read 1 and C in read 2.</td></tr><tr><td>REF_TOTAL_BASES</td><td>The total number of reference alleles observed</td></tr><tr><td>ALT_NONOXO_BASES</td><td>The count of observed A basecalls at C reference positions and T basecalls
 at G reference bases that are correlated to instrument read number in a way
 that rules out oxidation as the cause</td></tr><tr><td>ALT_OXO_BASES</td><td>The count of observed A basecalls at C reference positions and T basecalls
 at G reference bases that are correlated to instrument read number in a way
 that is consistent with oxidative damage.</td></tr><tr><td>OXIDATION_ERROR_RATE</td><td>The oxo error rate, calculated as max(ALT_OXO_BASES - ALT_NONOXO_BASES, 1) / TOTAL_BASES</td></tr><tr><td>OXIDATION_Q</td><td>-10 * log10(OXIDATION_ERROR_RATE)</td></tr><tr><td>C_REF_REF_BASES</td><td>The number of ref basecalls observed at sites where the genome reference == C.</td></tr><tr><td>G_REF_REF_BASES</td><td>The number of ref basecalls observed at sites where the genome reference == G.</td></tr><tr><td>C_REF_ALT_BASES</td><td>The number of alt (A/T) basecalls observed at sites where the genome reference == C.</td></tr><tr><td>G_REF_ALT_BASES</td><td>The number of alt (A/T) basecalls observed at sites where the genome reference == G.</td></tr><tr><td>C_REF_OXO_ERROR_RATE</td><td>The rate at which C>A and G>T substitutions are observed at C reference sites above the expected rate if there
 were no bias between sites with a C reference base vs. a G reference base.</td></tr><tr><td>C_REF_OXO_Q</td><td>C_REF_OXO_ERROR_RATE expressed as a phred-scaled quality score.</td></tr><tr><td>G_REF_OXO_ERROR_RATE</td><td>The rate at which C>A and G>T substitutions are observed at G reference sites above the expected rate if there
 were no bias between sites with a C reference base vs. a G reference base.</td></tr><tr><td>G_REF_OXO_Q</td><td>G_REF_OXO_ERROR_RATE expressed as a phred-scaled quality score.</td></tr></table>
</section>
<a id="CollectQualityYieldMetrics.QualityYieldMetrics"></a>
<h2>CollectQualityYieldMetrics.QualityYieldMetrics</h2>
<section>
<p>A set of metrics used to describe the general quality of a BAM file</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>TOTAL_READS</td><td>The total number of reads in the input file</td></tr><tr><td>PF_READS</td><td>The number of reads that are PF - pass filter</td></tr><tr><td>READ_LENGTH</td><td>The average read length of all the reads (will be fixed for a lane)</td></tr><tr><td>TOTAL_BASES</td><td>The total number of bases in all reads</td></tr><tr><td>PF_BASES</td><td>The total number of bases in all PF reads</td></tr><tr><td>Q20_BASES</td><td>The number of bases in all reads that achieve quality score 20 or higher</td></tr><tr><td>PF_Q20_BASES</td><td>The number of bases in PF reads that achieve quality score 20 or higher</td></tr><tr><td>Q30_BASES</td><td>The number of bases in all reads that achieve quality score 20 or higher</td></tr><tr><td>PF_Q30_BASES</td><td>The number of bases in PF reads that achieve quality score 20 or higher</td></tr><tr><td>Q20_EQUIVALENT_YIELD</td><td>The sum of quality scores of all bases divided by 20</td></tr><tr><td>PF_Q20_EQUIVALENT_YIELD</td><td>The sum of quality scores of all bases divided by 20</td></tr></table>
</section>
<a id="CollectRawWgsMetrics.RawWgsMetrics"></a>
<h2>CollectRawWgsMetrics.RawWgsMetrics</h2>
<section>
<p></p>
<table>
<tr><th>Field</th><th>Description</th></tr>
</table>
</section>
<a id="CollectVariantCallingMetrics.VariantCallingDetailMetrics"></a>
<h2>CollectVariantCallingMetrics.VariantCallingDetailMetrics</h2>
<section>
<p>A collection of metrics relating to snps and indels within a variant-calling file (VCF) for a given sample.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE_ALIAS</td><td>The name of the sample being assayed</td></tr><tr><td>HET_HOMVAR_RATIO</td><td>(count of hets)/(count of homozygous non-ref) for this sample</td></tr></table>
</section>
<a id="CollectVariantCallingMetrics.VariantCallingSummaryMetrics"></a>
<h2>CollectVariantCallingMetrics.VariantCallingSummaryMetrics</h2>
<section>
<p>A collection of metrics relating to snps and indels within a variant-calling file (VCF).</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>TOTAL_SNPS</td><td>The number of high confidence SNPs calls (i.e. non-reference genotypes) that were examined</td></tr><tr><td>NUM_IN_DB_SNP</td><td>The number of high confidence SNPs found in dbSNP</td></tr><tr><td>NOVEL_SNPS</td><td>The number of high confidence SNPS called that were not found in dbSNP</td></tr><tr><td>FILTERED_SNPS</td><td>The number of SNPs that are also filtered</td></tr><tr><td>PCT_DBSNP</td><td>The percentage of high confidence SNPs in dbSNP</td></tr><tr><td>DBSNP_TITV</td><td>The Transition/Transversion ratio of the SNP calls made at dbSNP sites</td></tr><tr><td>NOVEL_TITV</td><td>The Transition/Transversion ratio of the SNP calls made at non-dbSNP sites</td></tr><tr><td>TOTAL_INDELS</td><td>The number of high confidence Indel calls that were examined</td></tr><tr><td>NOVEL_INDELS</td><td>The number of high confidence Indels called that were not found in dbSNP</td></tr><tr><td>FILTERED_INDELS</td><td>The number of indels that are also filtered</td></tr><tr><td>PCT_DBSNP_INDELS</td><td>The percentage of high confidence Indels in dbSNP</td></tr><tr><td>NUM_IN_DB_SNP_INDELS</td><td>The number of high confidence Indels found in dbSNP</td></tr><tr><td>DBSNP_INS_DEL_RATIO</td><td>The Insertion/Deletion ratio of the Indel calls made at dbSNP sites</td></tr><tr><td>NOVEL_INS_DEL_RATIO</td><td>The Insertion/Deletion ratio of the Indel calls made at non-dbSNP sites</td></tr><tr><td>TOTAL_MULTIALLELIC_SNPS</td><td>The number of high confidence multiallelic SNP calls that were examined</td></tr><tr><td>NUM_IN_DB_SNP_MULTIALLELIC</td><td>The number of high confidence multiallelic SNPs found in dbSNP</td></tr><tr><td>TOTAL_COMPLEX_INDELS</td><td>The number of high confidence complex Indel calls that were examined</td></tr><tr><td>NUM_IN_DB_SNP_COMPLEX_INDELS</td><td>The number of high confidence complex Indels found in dbSNP</td></tr><tr><td>SNP_REFERENCE_BIAS</td><td>The rate at which reference bases are observed at ref/alt heterozygous SNP sites.</td></tr><tr><td>NUM_SINGLETONS</td><td>For summary metrics, the number of variants that appear in only one sample.
 For detail metrics, the number of variants that appear only in the current sample.</td></tr></table>
</section>
<a id="CollectWgsMetrics.WgsMetrics"></a>
<h2>CollectWgsMetrics.WgsMetrics</h2>
<section>
<p>Metrics for evaluating the performance of whole genome sequencing experiments.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>GENOME_TERRITORY</td><td>The number of non-N bases in the genome reference over which coverage will be evaluated.</td></tr><tr><td>MEAN_COVERAGE</td><td>The mean coverage in bases of the genome territory, after all filters are applied.</td></tr><tr><td>SD_COVERAGE</td><td>The standard deviation of coverage of the genome after all filters are applied.</td></tr><tr><td>MEDIAN_COVERAGE</td><td>The median coverage in bases of the genome territory, after all filters are applied.</td></tr><tr><td>MAD_COVERAGE</td><td>The median absolute deviation of coverage of the genome after all filters are applied.</td></tr><tr><td>PCT_EXC_MAPQ</td><td>The fraction of aligned bases that were filtered out because they were in reads with low mapping quality (default is < 20).</td></tr><tr><td>PCT_EXC_DUPE</td><td>The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</td></tr><tr><td>PCT_EXC_UNPAIRED</td><td>The fraction of aligned bases that were filtered out because they were in reads without a mapped mate pair.</td></tr><tr><td>PCT_EXC_BASEQ</td><td>The fraction of aligned bases that were filtered out because they were of low base quality (default is < 20).</td></tr><tr><td>PCT_EXC_OVERLAP</td><td>The fraction of aligned bases that were filtered out because they were the second observation from an insert with overlapping reads.</td></tr><tr><td>PCT_EXC_CAPPED</td><td>The fraction of aligned bases that were filtered out because they would have raised coverage above the capped value (default cap = 250x).</td></tr><tr><td>PCT_EXC_TOTAL</td><td>The total fraction of aligned bases excluded due to all filters.</td></tr><tr><td>PCT_5X</td><td>The fraction of bases that attained at least 5X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_10X</td><td>The fraction of bases that attained at least 10X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_15X</td><td>The fraction of bases that attained at least 15X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_20X</td><td>The fraction of bases that attained at least 20X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_25X</td><td>The fraction of bases that attained at least 25X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_30X</td><td>The fraction of bases that attained at least 30X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_40X</td><td>The fraction of bases that attained at least 40X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_50X</td><td>The fraction of bases that attained at least 50X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_60X</td><td>The fraction of bases that attained at least 60X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_70X</td><td>The fraction of bases that attained at least 70X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_80X</td><td>The fraction of bases that attained at least 80X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_90X</td><td>The fraction of bases that attained at least 90X sequence coverage in post-filtering bases.</td></tr><tr><td>PCT_100X</td><td>The fraction of bases that attained at least 100X sequence coverage in post-filtering bases.</td></tr></table>
</section>
<a id="CollectWgsMetricsFromQuerySorted.QuerySortedSeqMetrics"></a>
<h2>CollectWgsMetricsFromQuerySorted.QuerySortedSeqMetrics</h2>
<section>
<p>Metrics for evaluating the performance of whole genome sequencing experiments.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>TOTAL_BASES</td><td>The total number of bases, before any filters are applied.</td></tr><tr><td>TOTAL_USABLE_BASES</td><td>The number of usable bases, after all filters are applied.</td></tr><tr><td>TOTAL_READ_PAIRS</td><td>The number of read pairs, before all filters are applied.</td></tr><tr><td>TOTAL_DUPE_PAIRS</td><td>The number of duplicate read pairs, before all filters are applied.</td></tr><tr><td>TOTAL_ORIENTED_PAIRS</td><td>The number of read pairs with standard orientations from which to calculate mean insert size, after filters are applied.</td></tr><tr><td>MEAN_INSERT_SIZE</td><td>The mean insert size, after filters are applied.</td></tr></table>
</section>
<a id="CollectWgsMetricsFromSampledSites.SampledWgsMetrics"></a>
<h2>CollectWgsMetricsFromSampledSites.SampledWgsMetrics</h2>
<section>
<p></p>
<table>
<tr><th>Field</th><th>Description</th></tr>
</table>
</section>
<a id="DuplicationMetrics"></a>
<h2>DuplicationMetrics</h2>
<section>
<p>Metrics that are calculated during the process of marking duplicates
 within a stream of SAMRecords.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>LIBRARY</td><td>The library on which the duplicate marking was performed.</td></tr><tr><td>UNPAIRED_READS_EXAMINED</td><td>The number of mapped reads examined which did not have a mapped mate pair,
 either because the read is unpaired, or the read is paired to an unmapped mate.</td></tr><tr><td>READ_PAIRS_EXAMINED</td><td>The number of mapped read pairs examined.</td></tr><tr><td>UNMAPPED_READS</td><td>The total number of unmapped reads examined.</td></tr><tr><td>UNPAIRED_READ_DUPLICATES</td><td>The number of fragments that were marked as duplicates.</td></tr><tr><td>READ_PAIR_DUPLICATES</td><td>The number of read pairs that were marked as duplicates.</td></tr><tr><td>READ_PAIR_OPTICAL_DUPLICATES</td><td>The number of read pairs duplicates that were caused by optical duplication.
 Value is always < READ_PAIR_DUPLICATES, which counts all duplicates regardless of source.</td></tr><tr><td>PERCENT_DUPLICATION</td><td>The percentage of mapped sequence that is marked as duplicate.</td></tr><tr><td>ESTIMATED_LIBRARY_SIZE</td><td>The estimated number of unique molecules in the library based on PE duplication.</td></tr></table>
</section>
<a id="ExtractIlluminaBarcodes.BarcodeMetric"></a>
<h2>ExtractIlluminaBarcodes.BarcodeMetric</h2>
<section>
<p>Metrics produced by the ExtractIlluminaBarcodes program that is used to parse data in
 the basecalls directory and determine to which barcode each read should be assigned.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>BARCODE</td><td>The barcode (from the set of expected barcodes) for which the following metrics apply.
 Note that the "symbolic" barcode of NNNNNN is used to report metrics for all reads that
 do not match a barcode.</td></tr><tr><td>BARCODE_NAME</td><td></td></tr><tr><td>LIBRARY_NAME</td><td></td></tr><tr><td>READS</td><td>The total number of reads matching the barcode.</td></tr><tr><td>PF_READS</td><td>The number of PF reads matching this barcode (always less than or equal to READS).</td></tr><tr><td>PERFECT_MATCHES</td><td>The number of all reads matching this barcode that matched with 0 errors or no-calls.</td></tr><tr><td>PF_PERFECT_MATCHES</td><td>The number of PF reads matching this barcode that matched with 0 errors or no-calls.</td></tr><tr><td>ONE_MISMATCH_MATCHES</td><td>The number of all reads matching this barcode that matched with 1 error or no-call.</td></tr><tr><td>PF_ONE_MISMATCH_MATCHES</td><td>The number of PF reads matching this barcode that matched with 1 error or no-call.</td></tr><tr><td>PCT_MATCHES</td><td>The percentage of all reads in the lane that matched to this barcode.</td></tr><tr><td>RATIO_THIS_BARCODE_TO_BEST_BARCODE_PCT</td><td>The rate of all reads matching this barcode to all reads matching the most prevelant barcode. For the
 most prevelant barcode this will be 1, for all others it will be less than 1 (except for the possible
 exception of when there are more orphan reads than for any other barcode, in which case the value
 may be arbitrarily large).  One over the lowest number in this column gives you the fold-difference
 in representation between barcodes.</td></tr><tr><td>PF_PCT_MATCHES</td><td>The percentage of PF reads in the lane that matched to this barcode.</td></tr><tr><td>PF_RATIO_THIS_BARCODE_TO_BEST_BARCODE_PCT</td><td>The rate of PF reads matching this barcode to PF reads matching the most prevelant barcode. For the
 most prevelant barcode this will be 1, for all others it will be less than 1 (except for the possible
 exception of when there are more orphan reads than for any other barcode, in which case the value
 may be arbitrarily large).  One over the lowest number in this column gives you the fold-difference
 in representation of PF reads between barcodes.</td></tr><tr><td>PF_NORMALIZED_MATCHES</td><td>The "normalized" matches to each barcode. This is calculated as the number of pf reads matching
 this barcode over the sum of all pf reads matching any barcode (excluding orphans). If all barcodes
 are represented equally this will be 1.</td></tr></table>
</section>
<a id="GcBiasDetailMetrics"></a>
<h2>GcBiasDetailMetrics</h2>
<section>
<p>Class that holds detailed metrics about reads that fall within windows of a certain
 GC bin on the reference genome.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>ACCUMULATION_LEVEL</td><td></td></tr><tr><td>GC</td><td>The G+C content of the reference sequence represented by this bin. Values are from 0% to 100%</td></tr><tr><td>WINDOWS</td><td>The number of windows on the reference genome that have this G+C content.</td></tr><tr><td>READ_STARTS</td><td>The number of reads whose start position is at the start of a window of this GC.</td></tr><tr><td>MEAN_BASE_QUALITY</td><td>The mean quality (determined via the error rate) of all bases of all reads that are assigned to windows of this GC.</td></tr><tr><td>NORMALIZED_COVERAGE</td><td>The ration of "coverage" in this GC bin vs. the mean coverage of all GC bins. A number of
 1 represents mean coverage, a number less than one represents lower than mean coverage (e.g. 0.5
 means half as much coverage as average) while a number greater than one represents higher than
 mean coverage (e.g. 3.1 means this GC bin has 3.1 times more reads per window than average).</td></tr><tr><td>ERROR_BAR_WIDTH</td><td>The radius of error bars in this bin based on the number of observations made. For example if
 the normalized coverage is 0.75 and the error bar width is 0.1 then the error bars would be
 drawn from 0.65 to 0.85.</td></tr></table>
</section>
<a id="GcBiasMetrics"></a>
<h2>GcBiasMetrics</h2>
<section>
<p></p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>DETAILS</td><td></td></tr><tr><td>SUMMARY</td><td></td></tr></table>
</section>
<a id="GcBiasSummaryMetrics"></a>
<h2>GcBiasSummaryMetrics</h2>
<section>
<p>High level metrics that capture how biased the coverage in a certain lane is.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>ACCUMULATION_LEVEL</td><td></td></tr><tr><td>WINDOW_SIZE</td><td>The window size on the genome used to calculate the GC of the sequence.</td></tr><tr><td>TOTAL_CLUSTERS</td><td>The total number of clusters that were seen in the gc bias calculation.</td></tr><tr><td>ALIGNED_READS</td><td>The total number of aligned reads used to compute the gc bias metrics.</td></tr><tr><td>AT_DROPOUT</td><td>Illumina-style AT dropout metric.  Calculated by taking each GC bin independently and calculating
 (%ref_at_gc - %reads_at_gc) and summing all positive values for GC=[0..50].</td></tr><tr><td>GC_DROPOUT</td><td>Illumina-style GC dropout metric.  Calculated by taking each GC bin independently and calculating
 (%ref_at_gc - %reads_at_gc) and summing all positive values for GC=[50..100].</td></tr></table>
</section>
<a id="GenotypeConcordanceContingencyMetrics"></a>
<h2>GenotypeConcordanceContingencyMetrics</h2>
<section>
<p>Class that holds metrics about the Genotype Concordance contingency tables.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>VARIANT_TYPE</td><td>The type of the event (i.e. either SNP or INDEL)</td></tr><tr><td>TRUTH_SAMPLE</td><td>The name of the 'truth' sample</td></tr><tr><td>CALL_SAMPLE</td><td>The name of the 'call' sample</td></tr><tr><td>TP_COUNT</td><td>The TP (true positive) count across all variants</td></tr><tr><td>TN_COUNT</td><td>The TN (true negative) count across all variants</td></tr><tr><td>FP_COUNT</td><td>The FP (false positive) count across all variants</td></tr><tr><td>FN_COUNT</td><td>The FN (false negative) count across all variants</td></tr><tr><td>EMPTY_COUNT</td><td>The empty (no contingency info) count across all variants</td></tr></table>
</section>
<a id="GenotypeConcordanceDetailMetrics"></a>
<h2>GenotypeConcordanceDetailMetrics</h2>
<section>
<p>Class that holds detail metrics about Genotype Concordance</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>VARIANT_TYPE</td><td>The type of the event (i.e. either SNP or INDEL)</td></tr><tr><td>TRUTH_SAMPLE</td><td>The name of the 'truth' sample</td></tr><tr><td>CALL_SAMPLE</td><td>The name of the 'call' sample</td></tr><tr><td>TRUTH_STATE</td><td>The state of the 'truth' sample (i.e. HOM_REF, HET_REF_VAR1, HET_VAR1_VAR2...)</td></tr><tr><td>CALL_STATE</td><td>The state of the 'call' sample (i.e. HOM_REF, HET_REF_VAR1...)</td></tr><tr><td>COUNT</td><td>The number of events of type TRUTH_STATE and CALL_STATE for the EVENT_TYPE and SAMPLEs</td></tr><tr><td>CONTINGENCY_VALUES</td><td>The list of contingency table values (TP, TN, FP, FN) that are deduced from the truth/call state comparison, given the reference.

 In general, we are comparing two sets of alleles.  Therefore, we can have zero or more contingency table values represented in one comparison.  For example, if the truthset is
 a heterozygous call with both alleles non-reference (HET_VAR1_VAR2), and the callset is a heterozygous call with both alleles non-reference with one of the alternate alleles
 matching an alternate allele in the callset, we would have a true positive, false positive, and false negative.  The true positive is from the matching alternate alleles, the
 false positive is the alternate allele found in the callset but not found in the truthset, and the false negative is the alternate in the truthset not found in the callset.

 We also include a true negative in cases where the reference allele is found in both the truthset and callset.</td></tr></table>
</section>
<a id="GenotypeConcordanceSummaryMetrics"></a>
<h2>GenotypeConcordanceSummaryMetrics</h2>
<section>
<p>Class that holds summary metrics about Genotype Concordance</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>VARIANT_TYPE</td><td>The type of the event (i.e. either SNP or INDEL)</td></tr><tr><td>TRUTH_SAMPLE</td><td>The name of the 'truth' sample</td></tr><tr><td>CALL_SAMPLE</td><td>The name of the 'call' sample</td></tr><tr><td>HET_SENSITIVITY</td><td>The sensitivity for all heterozygous variants (Sensitivity is TP / (TP + FN))</td></tr><tr><td>HET_PPV</td><td>The ppv (positive predictive value) for all heterozygous variants (PPV is the TP / (TP + FP))</td></tr><tr><td>HET_SPECIFICITY</td><td>The specificity for all heterozygous variants cannot be calculated</td></tr><tr><td>HOMVAR_SENSITIVITY</td><td>The sensitivity for all homozygous variants (Sensitivity is TP / (TP + FN))</td></tr><tr><td>HOMVAR_PPV</td><td>The ppv (positive predictive value) for all homozygous variants (PPV is the TP / (TP + FP))</td></tr><tr><td>HOMVAR_SPECIFICITY</td><td>The specificity for all homozygous variants cannot be calculated.</td></tr><tr><td>VAR_SENSITIVITY</td><td>The sensitivity for all (heterozygous and homozygous) variants (Sensitivity is TP / (TP + FN))</td></tr><tr><td>VAR_PPV</td><td>The ppv (positive predictive value) for all (heterozygous and homozygous) variants (PPV is the TP / (TP + FP))</td></tr><tr><td>VAR_SPECIFICITY</td><td>The specificity for all (heterozygous and homozygous) variants (Specificity is TN / (FP + TN))</td></tr><tr><td>GENOTYPE_CONCORDANCE</td><td>The genotype concordance for all possible states.
  Genotype Concordance is the number of times the truth and call states match exactly / all truth and call combinations made</td></tr><tr><td>NON_REF_GENOTYPE_CONCORDANCE</td><td>The non-ref genotype concordance, ie for all var states only.
 Non Ref Genotype Concordance is the number of times the truth and call states match exactly for *vars only* / all truth and call *var* combinations made</td></tr></table>
</section>
<a id="HsMetrics"></a>
<h2>HsMetrics</h2>
<section>
<p>The set of metrics captured that are specific to a hybrid selection analysis.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>BAIT_SET</td><td>The name of the bait set used in the hybrid selection.</td></tr><tr><td>GENOME_SIZE</td><td>The number of bases in the reference genome used for alignment.</td></tr><tr><td>BAIT_TERRITORY</td><td>The number of bases which have one or more baits on top of them.</td></tr><tr><td>TARGET_TERRITORY</td><td>The unique number of target bases in the experiment where target is usually exons etc.</td></tr><tr><td>BAIT_DESIGN_EFFICIENCY</td><td>Target terrirtoy / bait territory.  1 == perfectly efficient, 0.5 = half of baited bases are not target.</td></tr><tr><td>TOTAL_READS</td><td>The total number of reads in the SAM or BAM file examine.</td></tr><tr><td>PF_READS</td><td>The number of reads that pass the vendor's filter.</td></tr><tr><td>PF_UNIQUE_READS</td><td>The number of PF reads that are not marked as duplicates.</td></tr><tr><td>PCT_PF_READS</td><td>PF reads / total reads.  The percent of reads passing filter.</td></tr><tr><td>PCT_PF_UQ_READS</td><td>PF Unique Reads / Total Reads.</td></tr><tr><td>PF_UQ_READS_ALIGNED</td><td>The number of PF unique reads that are aligned with mapping score > 0 to the reference genome.</td></tr><tr><td>PCT_PF_UQ_READS_ALIGNED</td><td>PF Reads Aligned / PF Reads.</td></tr><tr><td>PF_UQ_BASES_ALIGNED</td><td>The number of bases in the PF aligned reads that are mapped to a reference base. Accounts for clipping and gaps.</td></tr><tr><td>ON_BAIT_BASES</td><td>The number of PF aligned bases that mapped to a baited region of the genome.</td></tr><tr><td>NEAR_BAIT_BASES</td><td>The number of PF aligned bases that mapped to within a fixed interval of a baited region, but not on a baited region.</td></tr><tr><td>OFF_BAIT_BASES</td><td>The number of PF aligned bases that mapped to neither on or near a bait.</td></tr><tr><td>ON_TARGET_BASES</td><td>The number of PF aligned bases that mapped to a targeted region of the genome.</td></tr><tr><td>PCT_SELECTED_BASES</td><td>On+Near Bait Bases / PF Bases Aligned.</td></tr><tr><td>PCT_OFF_BAIT</td><td>The percentage of aligned PF bases that mapped neither on or near a bait.</td></tr><tr><td>ON_BAIT_VS_SELECTED</td><td>The percentage of on+near bait bases that are on as opposed to near.</td></tr><tr><td>MEAN_BAIT_COVERAGE</td><td>The mean coverage of all baits in the experiment.</td></tr><tr><td>MEAN_TARGET_COVERAGE</td><td>The mean coverage of targets that received at least coverage depth = 2 at one base.</td></tr><tr><td>PCT_USABLE_BASES_ON_BAIT</td><td>The number of aligned, de-duped, on-bait bases out of the PF bases available.</td></tr><tr><td>PCT_USABLE_BASES_ON_TARGET</td><td>The number of aligned, de-duped, on-target bases out of the PF bases available.</td></tr><tr><td>FOLD_ENRICHMENT</td><td>The fold by which the baited region has been amplified above genomic background.</td></tr><tr><td>ZERO_CVG_TARGETS_PCT</td><td>The number of targets that did not reach coverage=2 over any base.</td></tr><tr><td>FOLD_80_BASE_PENALTY</td><td>The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.</td></tr><tr><td>PCT_TARGET_BASES_2X</td><td>The percentage of ALL target bases achieving 2X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_10X</td><td>The percentage of ALL target bases achieving 10X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_20X</td><td>The percentage of ALL target bases achieving 20X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_30X</td><td>The percentage of ALL target bases achieving 30X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_40X</td><td>The percentage of ALL target bases achieving 40X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_50X</td><td>The percentage of ALL target bases achieving 50X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_100X</td><td>The percentage of ALL target bases achieving 100X or greater coverage.</td></tr><tr><td>HS_LIBRARY_SIZE</td><td>The estimated number of unique molecules in the selected part of the library.</td></tr><tr><td>HS_PENALTY_10X</td><td>The "hybrid selection penalty" incurred to get 80% of target bases to 10X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 10X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 10 * HS_PENALTY_10X.</td></tr><tr><td>HS_PENALTY_20X</td><td>The "hybrid selection penalty" incurred to get 80% of target bases to 20X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 20X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 20 * HS_PENALTY_20X.</td></tr><tr><td>HS_PENALTY_30X</td><td>The "hybrid selection penalty" incurred to get 80% of target bases to 30X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 30X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 30 * HS_PENALTY_30X.</td></tr><tr><td>HS_PENALTY_40X</td><td>The "hybrid selection penalty" incurred to get 80% of target bases to 40X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 40X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 40 * HS_PENALTY_40X.</td></tr><tr><td>HS_PENALTY_50X</td><td>The "hybrid selection penalty" incurred to get 80% of target bases to 50X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 50X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 50 * HS_PENALTY_50X.</td></tr><tr><td>HS_PENALTY_100X</td><td>The "hybrid selection penalty" incurred to get 80% of target bases to 100X. This metric
 should be interpreted as: if I have a design with 10 megabases of target, and want to get
 100X coverage I need to sequence until PF_ALIGNED_BASES = 10^7 * 100 * HS_PENALTY_100X.</td></tr><tr><td>AT_DROPOUT</td><td>A measure of how undercovered <= 50% GC regions are relative to the mean. For each GC bin [0..50]
 we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
 AT DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
 reads that should have mapped to GC<=50% regions mapped elsewhere.</td></tr><tr><td>GC_DROPOUT</td><td>A measure of how undercovered >= 50% GC regions are relative to the mean. For each GC bin [50..100]
 we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
 GC DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
 reads that should have mapped to GC>=50% regions mapped elsewhere.</td></tr></table>
</section>
<a id="IlluminaBasecallingMetrics"></a>
<h2>IlluminaBasecallingMetrics</h2>
<section>
<p>Metric for Illumina Basecalling that stores means and standard deviations on a per-barcode per-lane basis.  Averages
 and means are taken over all tiles.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>LANE</td><td>The lane for which the metrics were calculated.</td></tr><tr><td>MOLECULAR_BARCODE_SEQUENCE_1</td><td>The barcode sequence for which the metrics were calculated.</td></tr><tr><td>MOLECULAR_BARCODE_NAME</td><td>The barcode name for which the metrics were calculated.</td></tr><tr><td>TOTAL_BASES</td><td>The total number of bases assigned to the index.</td></tr><tr><td>PF_BASES</td><td>The total number of passing-filter bases assigned to the index.</td></tr><tr><td>TOTAL_READS</td><td>The total number of reads assigned to the index.</td></tr><tr><td>PF_READS</td><td>The total number of passing-filter reads assigned to the index.</td></tr><tr><td>TOTAL_CLUSTERS</td><td>The total number of clusters assigned to the index.</td></tr><tr><td>PF_CLUSTERS</td><td>The total number of PF clusters assigned to the index.</td></tr><tr><td>MEAN_CLUSTERS_PER_TILE</td><td>The mean number of clusters per tile.</td></tr><tr><td>SD_CLUSTERS_PER_TILE</td><td>The standard deviation of clusters per tile.</td></tr><tr><td>MEAN_PCT_PF_CLUSTERS_PER_TILE</td><td>The mean percentage of pf clusters per tile.</td></tr><tr><td>SD_PCT_PF_CLUSTERS_PER_TILE</td><td>The standard deviation in percentage of pf clusters per tile.</td></tr><tr><td>MEAN_PF_CLUSTERS_PER_TILE</td><td>The mean number of pf clusters per tile.</td></tr><tr><td>SD_PF_CLUSTERS_PER_TILE</td><td>The standard deviation in number of pf clusters per tile.</td></tr></table>
</section>
<a id="IlluminaLaneMetrics"></a>
<h2>IlluminaLaneMetrics</h2>
<section>
<p>Embodies characteristics that describe a lane.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>CLUSTER_DENSITY</td><td>The number of clusters per unit area on the this lane expressed in units of [cluster / mm^2].</td></tr><tr><td>LANE</td><td>This lane's number.</td></tr></table>
</section>
<a id="IlluminaPhasingMetrics"></a>
<h2>IlluminaPhasingMetrics</h2>
<section>
<p>Metrics for Illumina Basecalling that stores median phasing and prephasing percentages on a per-template-read, per-lane basis.
 For each lane/template read # (i.e. FIRST, SECOND) combination we will store the median values of both the phasing and prephasing
 values for every tile in that lane/template read pair.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>LANE</td><td></td></tr><tr><td>TYPE_NAME</td><td></td></tr><tr><td>PHASING_APPLIED</td><td></td></tr><tr><td>PREPHASING_APPLIED</td><td></td></tr></table>
</section>
<a id="InsertSizeMetrics"></a>
<h2>InsertSizeMetrics</h2>
<section>
<p>Metrics about the insert size distribution of a paired-end library, created by the
 CollectInsertSizeMetrics program and usually written to a file with the extension
 ".insert_size_metrics".  In addition the insert size distribution is plotted to
 a file with the extension ".insert_size_Histogram.pdf".</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>MEDIAN_INSERT_SIZE</td><td>The MEDIAN insert size of all paired end reads where both ends mapped to the same chromosome.</td></tr><tr><td>MEDIAN_ABSOLUTE_DEVIATION</td><td>The median absolute deviation of the distribution.  If the distribution is essentially normal then
 the standard deviation can be estimated as ~1.4826 * MAD.</td></tr><tr><td>MIN_INSERT_SIZE</td><td>The minimum measured insert size.  This is usually 1 and not very useful as it is likely artifactual.</td></tr><tr><td>MAX_INSERT_SIZE</td><td>The maximum measure insert size by alignment. This is usually very high representing either an artifact
 or possibly the presence of a structural re-arrangement.</td></tr><tr><td>MEAN_INSERT_SIZE</td><td>The mean insert size of the "core" of the distribution. Artefactual outliers in the distribution often
 cause calculation of nonsensical mean and stdev values.  To avoid this the distribution is first trimmed
 to a "core" distribution of +/- N median absolute deviations around the median insert size. By default
 N=10, but this is configurable.</td></tr><tr><td>STANDARD_DEVIATION</td><td>Standard deviation of insert sizes over the "core" of the distribution.</td></tr><tr><td>READ_PAIRS</td><td>The total number of read pairs that were examined in the entire distribution.</td></tr><tr><td>PAIR_ORIENTATION</td><td>The pair orientation of the reads in this data category.</td></tr><tr><td>WIDTH_OF_10_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 10% of all read pairs.</td></tr><tr><td>WIDTH_OF_20_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 20% of all read pairs.</td></tr><tr><td>WIDTH_OF_30_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 30% of all read pairs.</td></tr><tr><td>WIDTH_OF_40_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 40% of all read pairs.</td></tr><tr><td>WIDTH_OF_50_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 50% of all read pairs.</td></tr><tr><td>WIDTH_OF_60_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 60% of all read pairs.</td></tr><tr><td>WIDTH_OF_70_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 70% of all read pairs.
 This metric divided by 2 should approximate the standard deviation when the insert size
 distribution is a normal distribution.</td></tr><tr><td>WIDTH_OF_80_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 80% of all read pairs.</td></tr><tr><td>WIDTH_OF_90_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 90% of all read pairs.</td></tr><tr><td>WIDTH_OF_99_PERCENT</td><td>The "width" of the bins, centered around the median, that encompass 100% of all read pairs.</td></tr></table>
</section>
<a id="JumpingLibraryMetrics"></a>
<h2>JumpingLibraryMetrics</h2>
<section>
<p>High level metrics about the presence of  outward- and inward-facing pairs
 within a SAM file generated with a jumping library, produced by
 the CollectJumpingLibraryMetrics program and usually stored in a file with
 the extension ".jump_metrics".</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>JUMP_PAIRS</td><td>The number of outward-facing pairs in the SAM file</td></tr><tr><td>JUMP_DUPLICATE_PAIRS</td><td>The number of outward-facing pairs that are duplicates</td></tr><tr><td>JUMP_DUPLICATE_PCT</td><td>The percentage of outward-facing pairs that are marked as duplicates</td></tr><tr><td>JUMP_LIBRARY_SIZE</td><td>The estimated library size for outward-facing pairs</td></tr><tr><td>JUMP_MEAN_INSERT_SIZE</td><td>The mean insert size for outward-facing pairs</td></tr><tr><td>JUMP_STDEV_INSERT_SIZE</td><td>The standard deviation on the insert size for outward-facing pairs</td></tr><tr><td>NONJUMP_PAIRS</td><td>The number of inward-facing pairs in the SAM file</td></tr><tr><td>NONJUMP_DUPLICATE_PAIRS</td><td>The number of inward-facing pais that are duplicates</td></tr><tr><td>NONJUMP_DUPLICATE_PCT</td><td>The percentage of inward-facing pairs that are marked as duplicates</td></tr><tr><td>NONJUMP_LIBRARY_SIZE</td><td>The estimated library size for inward-facing pairs</td></tr><tr><td>NONJUMP_MEAN_INSERT_SIZE</td><td>The mean insert size for inward-facing pairs</td></tr><tr><td>NONJUMP_STDEV_INSERT_SIZE</td><td>The standard deviation on the insert size for inward-facing pairs</td></tr><tr><td>CHIMERIC_PAIRS</td><td>The number of pairs where either (a) the ends fall on different chromosomes or (b) the insert size
 is greater than the maximum of 100000 or 2 times the mode of the insert size for outward-facing pairs.</td></tr><tr><td>FRAGMENTS</td><td>The number of fragments in the SAM file</td></tr><tr><td>PCT_JUMPS</td><td>The number of outward-facing pairs expressed as a percentage of the total of all outward facing pairs,
 inward-facing pairs, and chimeric pairs.</td></tr><tr><td>PCT_NONJUMPS</td><td>The number of inward-facing pairs expressed as a percentage of the total of all outward facing pairs,
 inward-facing pairs, and chimeric pairs.</td></tr><tr><td>PCT_CHIMERAS</td><td>The number of chimeric pairs expressed as a percentage of the total of all outward facing pairs,
 inward-facing pairs, and chimeric pairs.</td></tr></table>
</section>
<a id="MultilevelMetrics"></a>
<h2>MultilevelMetrics</h2>
<section>
<p></p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE</td><td>The sample to which these metrics apply.  If null, it means they apply
 to all reads in the file.</td></tr><tr><td>LIBRARY</td><td>The library to which these metrics apply.  If null, it means that the
 metrics were accumulated at the sample level.</td></tr><tr><td>READ_GROUP</td><td>The read group to which these metrics apply.  If null, it means that
 the metrics were accumulated at the library or sample level.</td></tr></table>
</section>
<a id="RnaSeqMetrics"></a>
<h2>RnaSeqMetrics</h2>
<section>
<p>Metrics about the alignment of RNA-seq reads within a SAM file to genes, produced by the CollectRnaSeqMetrics
 program and usually stored in a file with the extension ".rna_metrics".</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>PF_BASES</td><td>The total number of PF bases including non-aligned reads.</td></tr><tr><td>PF_ALIGNED_BASES</td><td>The total number of aligned PF bases.  Non-primary alignments are not counted. Bases in aligned reads that
 do not correspond to reference (e.g. soft clips, insertions) are not counted.</td></tr><tr><td>RIBOSOMAL_BASES</td><td>Number of bases in primary aligments that align to ribosomal sequence.</td></tr><tr><td>CODING_BASES</td><td>Number of bases in primary aligments that align to a non-UTR coding base for some gene, and not ribosomal sequence.</td></tr><tr><td>UTR_BASES</td><td>Number of bases in primary aligments that align to a UTR base for some gene, and not a coding base.</td></tr><tr><td>INTRONIC_BASES</td><td>Number of bases in primary aligments that align to an intronic base for some gene, and not a coding or UTR base.</td></tr><tr><td>INTERGENIC_BASES</td><td>Number of bases in primary aligments that do not align to any gene.</td></tr><tr><td>IGNORED_READS</td><td>Number of primary alignments that map to a sequence specified on command-line as IGNORED_SEQUENCE.  These are not
 counted in PF_ALIGNED_BASES, CORRECT_STRAND_READS, INCORRECT_STRAND_READS, or any of the base-counting metrics.
 These reads are counted in PF_BASES.</td></tr><tr><td>CORRECT_STRAND_READS</td><td>Number of aligned reads that map to the correct strand.  0 if library is not strand-specific.</td></tr><tr><td>INCORRECT_STRAND_READS</td><td>Number of aligned reads that map to the incorrect strand.  0 if library is not strand-specific.</td></tr><tr><td>PCT_RIBOSOMAL_BASES</td><td>RIBOSOMAL_BASES / PF_ALIGNED_BASES</td></tr><tr><td>PCT_CODING_BASES</td><td>CODING_BASES / PF_ALIGNED_BASES</td></tr><tr><td>PCT_UTR_BASES</td><td>UTR_BASES / PF_ALIGNED_BASES</td></tr><tr><td>PCT_INTRONIC_BASES</td><td>INTRONIC_BASES / PF_ALIGNED_BASES</td></tr><tr><td>PCT_INTERGENIC_BASES</td><td>INTERGENIC_BASES / PF_ALIGNED_BASES</td></tr><tr><td>PCT_MRNA_BASES</td><td>PCT_UTR_BASES + PCT_CODING_BASES</td></tr><tr><td>PCT_USABLE_BASES</td><td>The percentage of bases mapping to mRNA divided by the total number of PF bases.</td></tr><tr><td>PCT_CORRECT_STRAND_READS</td><td>CORRECT_STRAND_READS/(CORRECT_STRAND_READS + INCORRECT_STRAND_READS).  0 if library is not strand-specific.</td></tr><tr><td>MEDIAN_CV_COVERAGE</td><td>The median CV of coverage of the 1000 most highly expressed transcripts. Ideal value = 0.</td></tr><tr><td>MEDIAN_5PRIME_BIAS</td><td>The median 5 prime bias of the 1000 most highly expressed transcripts, where 5 prime bias is calculated per
 transcript as: mean coverage of the 5' most 100 bases divided by the mean coverage of the whole transcript.</td></tr><tr><td>MEDIAN_3PRIME_BIAS</td><td>The median 3 prime bias of the 1000 most highly expressed transcripts, where 3 prime bias is calculated per
 transcript as: mean coverage of the 3' most 100 bases divided by the mean coverage of the whole transcript.</td></tr><tr><td>MEDIAN_5PRIME_TO_3PRIME_BIAS</td><td>The ratio of coverage at the 5' end of to the 3' end based on the 1000 most highly expressed transcripts.</td></tr></table>
</section>
<a id="RrbsCpgDetailMetrics"></a>
<h2>RrbsCpgDetailMetrics</h2>
<section>
<p>Holds information about CpG sites encountered for RRBS processing QC</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SEQUENCE_NAME</td><td>Sequence the CpG is seen in</td></tr><tr><td>POSITION</td><td>Position within the sequence of the CpG site</td></tr><tr><td>TOTAL_SITES</td><td>Number of times this CpG site was encountered</td></tr><tr><td>CONVERTED_SITES</td><td>Number of times this CpG site was converted (TG for + strand, CA for - strand)</td></tr><tr><td>PCT_CONVERTED</td><td>TOTAL_BASES / CONVERTED_BASES</td></tr></table>
</section>
<a id="RrbsSummaryMetrics"></a>
<h2>RrbsSummaryMetrics</h2>
<section>
<p>Holds summary statistics from RRBS processing QC</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>READS_ALIGNED</td><td>Number of mapped reads processed</td></tr><tr><td>NON_CPG_BASES</td><td>Number of times a non-CpG cytosine was encountered</td></tr><tr><td>NON_CPG_CONVERTED_BASES</td><td>Number of times a non-CpG cytosine was converted (C->T for +, G->A for -)</td></tr><tr><td>PCT_NON_CPG_BASES_CONVERTED</td><td>NON_CPG_BASES / NON_CPG_CONVERTED_BASES</td></tr><tr><td>CPG_BASES_SEEN</td><td>Number of CpG sites encountered</td></tr><tr><td>CPG_BASES_CONVERTED</td><td>Number of CpG sites that were converted (TG for +, CA for -)</td></tr><tr><td>PCT_CPG_BASES_CONVERTED</td><td>CPG_BASES_SEEN / CPG_BASES_CONVERTED</td></tr><tr><td>MEAN_CPG_COVERAGE</td><td>Mean coverage of CpG sites</td></tr><tr><td>MEDIAN_CPG_COVERAGE</td><td>Median coverage of CpG sites</td></tr><tr><td>READS_WITH_NO_CPG</td><td>Number of reads discarded for having no CpG sites</td></tr><tr><td>READS_IGNORED_SHORT</td><td>Number of reads discarded due to being too short</td></tr><tr><td>READS_IGNORED_MISMATCHES</td><td>Number of reads discarded for exceeding the mismatch threshold</td></tr></table>
</section>
<a id="SamFileValidator.ValidationMetrics"></a>
<h2>SamFileValidator.ValidationMetrics</h2>
<section>
<p></p>
<table>
<tr><th>Field</th><th>Description</th></tr>
</table>
</section>
<a id="SequencingArtifactMetrics.BaitBiasDetailMetrics"></a>
<h2>SequencingArtifactMetrics.BaitBiasDetailMetrics</h2>
<section>
<p>Bait bias artifacts broken down by context.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE_ALIAS</td><td>The name of the sample being assayed.</td></tr><tr><td>LIBRARY</td><td>The name of the library being assayed.</td></tr><tr><td>REF_BASE</td><td>The (upper-case) original base on the reference strand.</td></tr><tr><td>ALT_BASE</td><td>The (upper-case) alternative base that is called as a result of DNA damage.</td></tr><tr><td>CONTEXT</td><td>The sequence context to which the analysis is constrained.</td></tr><tr><td>FWD_CXT_REF_BASES</td><td>The number of REF_BASE:REF_BASE alignments at sites with the given reference context.</td></tr><tr><td>FWD_CXT_ALT_BASES</td><td>The number of REF_BASE:ALT_BASE alignments at sites with the given reference context.</td></tr><tr><td>REV_CXT_REF_BASES</td><td>The number of ~REF_BASE:~REF_BASE alignments at sites complementary to the given reference context.</td></tr><tr><td>REV_CXT_ALT_BASES</td><td>The number of ~REF_BASE:~ALT_BASE alignments at sites complementary to the given reference context.</td></tr><tr><td>FWD_ERROR_RATE</td><td>The substitution rate of REF_BASE:ALT_BASE, calculated as max(1e-10, FWD_CXT_ALT_BASES / (FWD_CXT_ALT_BASES + FWD_CXT_REF_BASES)).</td></tr><tr><td>REV_ERROR_RATE</td><td>The substitution rate of ~REF_BASE:~ALT_BASE, calculated as max(1e-10, REV_CXT_ALT_BASES / (REV_CXT_ALT_BASES + REV_CXT_REF_BASES)).</td></tr><tr><td>ERROR_RATE</td><td>The bait bias error rate, calculated as max(1e-10, FWD_ERROR_RATE - REV_ERROR_RATE).</td></tr><tr><td>QSCORE</td><td>The Phred-scaled quality score of the artifact, calculated as -10 * log10(ERROR_RATE).</td></tr></table>
</section>
<a id="SequencingArtifactMetrics.BaitBiasSummaryMetrics"></a>
<h2>SequencingArtifactMetrics.BaitBiasSummaryMetrics</h2>
<section>
<p>Summary analysis of a single bait bias artifact, also known as a reference bias artifact.

 These artifacts occur during or after the target selection step, and correlate with substitution
 rates that are "biased", or higher for sites having one base on the reference/positive strand
 relative to sites having the complementary base on that strand.

 For example, a G>T artifact during the target selection step might result in a higher
 G>T / C>A substitution rate at sites with a G on the positive strand (and C on the negative),
 relative to sites with the flip (C positive / G negative). This is known as the "G-Ref" artifact.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE_ALIAS</td><td>The name of the sample being assayed.</td></tr><tr><td>LIBRARY</td><td>The name of the library being assayed.</td></tr><tr><td>REF_BASE</td><td>The (upper-case) original base on the reference strand.</td></tr><tr><td>ALT_BASE</td><td>The (upper-case) alternative base that is called as a result of DNA damage.</td></tr><tr><td>TOTAL_QSCORE</td><td>The total Phred-scaled Q-score for this artifact. A lower Q-score
 means a higher probability that a REF_BASE:ALT_BASE observation
 randomly picked from the data will be due to this artifact, rather
 than a true variant.</td></tr><tr><td>WORST_CXT</td><td>The sequence context (reference bases surrounding the locus of interest) having the lowest Q-score among all contexts for this artifact.</td></tr><tr><td>WORST_CXT_QSCORE</td><td>The Q-score for the worst context.</td></tr><tr><td>WORST_PRE_CXT</td><td>The pre-context (reference bases leading up to the locus of interest) with the lowest Q-score.</td></tr><tr><td>WORST_PRE_CXT_QSCORE</td><td>The Q-score for the worst pre-context.</td></tr><tr><td>WORST_POST_CXT</td><td>The post-context (reference bases trailing after the locus of interest) with the lowest Q-score.</td></tr><tr><td>WORST_POST_CXT_QSCORE</td><td>The Q-score for the worst post-context.</td></tr><tr><td>ARTIFACT_NAME</td><td>A "nickname" of this artifact, if it is a known error mode.</td></tr></table>
</section>
<a id="SequencingArtifactMetrics.PreAdapterDetailMetrics"></a>
<h2>SequencingArtifactMetrics.PreAdapterDetailMetrics</h2>
<section>
<p>Pre-adapter artifacts broken down by context.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE_ALIAS</td><td>The name of the sample being assayed.</td></tr><tr><td>LIBRARY</td><td>The name of the library being assayed.</td></tr><tr><td>REF_BASE</td><td>The (upper-case) original base on the reference strand.</td></tr><tr><td>ALT_BASE</td><td>The (upper-case) alternative base that is called as a result of DNA damage.</td></tr><tr><td>CONTEXT</td><td>The sequence context to which the analysis is constrained.</td></tr><tr><td>PRO_REF_BASES</td><td>The number of REF_BASE:REF_BASE alignments having a read number and orientation that supports the presence of this artifact.</td></tr><tr><td>PRO_ALT_BASES</td><td>The number of REF_BASE:ALT_BASE alignments having a read number and orientation that supports the presence of this artifact.</td></tr><tr><td>CON_REF_BASES</td><td>The number of REF_BASE:REF_BASE alignments having a read number and orientation that refutes the presence of this artifact.</td></tr><tr><td>CON_ALT_BASES</td><td>The number of REF_BASE:ALT_BASE alignments having a read number and orientation that refutes the presence of this artifact.</td></tr><tr><td>ERROR_RATE</td><td>The estimated error rate due to this artifact.
 Calculated as max(1e-10, (PRO_ALT_BASES - CON_ALT_BASES) / (PRO_ALT_BASES + PRO_REF_BASES + CON_ALT_BASES + CON_REF_BASES)).</td></tr><tr><td>QSCORE</td><td>The Phred-scaled quality score of the artifact, calculated as -10 * log10(ERROR_RATE).</td></tr></table>
</section>
<a id="SequencingArtifactMetrics.PreAdapterSummaryMetrics"></a>
<h2>SequencingArtifactMetrics.PreAdapterSummaryMetrics</h2>
<section>
<p>Summary analysis of a single pre-adapter artifact.

 These artifacts occur on the original template strand, before the addition of adapters,
 so they correlate with read number / orientation in a specific way.

 For example, the well-known "Oxo-G" artifact occurs when a G on the template
 strand is oxidized, giving it an affinity for binding to A rather than the usual C.
 Thus PCR will introduce apparent G>T substitutions in read 1 and C>A in read 2.
 In the resulting alignments, a given G>T or C>A observation could either be:

 1. a true mutation
 2. an OxoG artifact
 3. some other kind of artifact

 On average, we assume that 1 and 3 will not display this read number / orientation bias, so
 their contributions will cancel out in the calculation.</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>SAMPLE_ALIAS</td><td>The name of the sample being assayed.</td></tr><tr><td>LIBRARY</td><td>The name of the library being assayed.</td></tr><tr><td>REF_BASE</td><td>The (upper-case) original base on the reference strand.</td></tr><tr><td>ALT_BASE</td><td>The (upper-case) alternative base that is called as a result of DNA damage.</td></tr><tr><td>TOTAL_QSCORE</td><td>The total Phred-scaled Q-score for this artifact. A lower Q-score
 means a higher probability that a REF_BASE:ALT_BASE observation
 randomly picked from the data will be due to this artifact, rather
 than a true variant.</td></tr><tr><td>WORST_CXT</td><td>The sequence context (reference bases surrounding the locus of interest) having the lowest Q-score among all contexts for this artifact.</td></tr><tr><td>WORST_CXT_QSCORE</td><td>The Q-score for the worst context.</td></tr><tr><td>WORST_PRE_CXT</td><td>The pre-context (reference bases leading up to the locus of interest) with the lowest Q-score.</td></tr><tr><td>WORST_PRE_CXT_QSCORE</td><td>The Q-score for the worst pre-context.</td></tr><tr><td>WORST_POST_CXT</td><td>The post-context (reference bases trailing after the locus of interest) with the lowest Q-score.</td></tr><tr><td>WORST_POST_CXT_QSCORE</td><td>The Q-score for the worst post-context.</td></tr><tr><td>ARTIFACT_NAME</td><td>A "nickname" of this artifact, if it is a known error mode.</td></tr></table>
</section>
<a id="TargetedPcrMetrics"></a>
<h2>TargetedPcrMetrics</h2>
<section>
<p>Metrics class for targeted pcr runs such as TSCA runs</p>
<table>
<tr><th>Field</th><th>Description</th></tr>
<tr><td>CUSTOM_AMPLICON_SET</td><td>The name of the amplicon set used in this metrics collection run</td></tr><tr><td>GENOME_SIZE</td><td>The number of bases in the reference genome used for alignment.</td></tr><tr><td>AMPLICON_TERRITORY</td><td>The number of unique bases covered by the intervals of all amplicons in the amplicon set</td></tr><tr><td>TARGET_TERRITORY</td><td>The number of unique bases covered by the intervals of all targets that should be covered</td></tr><tr><td>TOTAL_READS</td><td>The total number of reads in the SAM or BAM file examine.</td></tr><tr><td>PF_READS</td><td>The number of reads that pass the vendor's filter.</td></tr><tr><td>PF_BASES</td><td>THe number of bases in the SAM or BAM file to be examined</td></tr><tr><td>PF_UNIQUE_READS</td><td>The number of PF reads that are not marked as duplicates.</td></tr><tr><td>PCT_PF_READS</td><td>PF reads / total reads.  The percent of reads passing filter.</td></tr><tr><td>PCT_PF_UQ_READS</td><td>PF Unique Reads / Total Reads.</td></tr><tr><td>PF_UQ_READS_ALIGNED</td><td>The number of PF unique reads that are aligned with mapping score > 0 to the reference genome.</td></tr><tr><td>PF_SELECTED_PAIRS</td><td>Tracks the number of read pairs that we see that are PF (used to calculate library size)</td></tr><tr><td>PF_SELECTED_UNIQUE_PAIRS</td><td>Tracks the number of unique PF reads pairs we see (used to calc library size)</td></tr><tr><td>PCT_PF_UQ_READS_ALIGNED</td><td>PF Reads Aligned / PF Reads.</td></tr><tr><td>PF_UQ_BASES_ALIGNED</td><td>The number of PF unique bases that are aligned with mapping score > 0 to the reference genome.</td></tr><tr><td>ON_AMPLICON_BASES</td><td>The number of PF aligned amplified that mapped to an amplified region of the genome.</td></tr><tr><td>NEAR_AMPLICON_BASES</td><td>The number of PF aligned bases that mapped to within a fixed interval of an amplified region, but not on a baited region.</td></tr><tr><td>OFF_AMPLICON_BASES</td><td>The number of PF aligned bases that mapped to neither on or near an amplicon.</td></tr><tr><td>ON_TARGET_BASES</td><td>The number of PF aligned bases that mapped to a targeted region of the genome.</td></tr><tr><td>ON_TARGET_FROM_PAIR_BASES</td><td>The number of PF aligned bases that are mapped in pair to a targeted region of the genome.</td></tr><tr><td>PCT_AMPLIFIED_BASES</td><td>On+Near Amplicon Bases / PF Bases Aligned.</td></tr><tr><td>PCT_OFF_AMPLICON</td><td>The percentage of aligned PF bases that mapped neither on or near an amplicon.</td></tr><tr><td>ON_AMPLICON_VS_SELECTED</td><td>The percentage of on+near amplicon bases that are on as opposed to near.</td></tr><tr><td>MEAN_AMPLICON_COVERAGE</td><td>The mean coverage of all amplicons in the experiment.</td></tr><tr><td>MEAN_TARGET_COVERAGE</td><td>The mean coverage of targets that recieved at least coverage depth = 2 at one base.</td></tr><tr><td>FOLD_ENRICHMENT</td><td>The fold by which the amplicon region has been amplified above genomic background.</td></tr><tr><td>ZERO_CVG_TARGETS_PCT</td><td>The number of targets that did not reach coverage=2 over any base.</td></tr><tr><td>FOLD_80_BASE_PENALTY</td><td>The fold over-coverage necessary to raise 80% of bases in "non-zero-cvg" targets to
 the mean coverage level in those targets.</td></tr><tr><td>PCT_TARGET_BASES_2X</td><td>The percentage of ALL target bases achieving 2X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_10X</td><td>The percentage of ALL target bases achieving 10X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_20X</td><td>The percentage of ALL target bases achieving 20X or greater coverage.</td></tr><tr><td>PCT_TARGET_BASES_30X</td><td>The percentage of ALL target bases achieving 30X or greater coverage.</td></tr><tr><td>AT_DROPOUT</td><td>A measure of how undercovered <= 50% GC regions are relative to the mean. For each GC bin [0..50]
 we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
 AT DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
 reads that should have mapped to GC<=50% regions mapped elsewhere.</td></tr><tr><td>GC_DROPOUT</td><td>A measure of how undercovered >= 50% GC regions are relative to the mean. For each GC bin [50..100]
 we calculate a = % of target territory, and b = % of aligned reads aligned to these targets.
 GC DROPOUT is then abs(sum(a-b when a-b < 0)). E.g. if the value is 5% this implies that 5% of total
 reads that should have mapped to GC>=50% regions mapped elsewhere.</td></tr></table>
</section>
