<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:41 EST 2015 -->
<TITLE>
AlignmentSummaryMetrics (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="AlignmentSummaryMetrics (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/AlignmentSummaryMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;PREV CLASS&nbsp;
&nbsp;<A HREF="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/analysis/AlignmentSummaryMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="AlignmentSummaryMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;<A HREF="#nested_class_summary">NESTED</A>&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.analysis</FONT>
<BR>
Class AlignmentSummaryMetrics</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../resources/inherit.gif" ALT="extended by ">htsjdk.samtools.metrics.MetricBase
      <IMG SRC="../../resources/inherit.gif" ALT="extended by "><A HREF="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">picard.metrics.MultilevelMetrics</A>
          <IMG SRC="../../resources/inherit.gif" ALT="extended by "><B>picard.analysis.AlignmentSummaryMetrics</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>AlignmentSummaryMetrics</B><DT>extends <A HREF="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</A></DL>
</PRE>

<P>
High level metrics about the alignment of reads within a SAM file, produced by
 the CollectAlignmentSummaryMetrics program and usually stored in a file with
 the extension ".alignment_summary_metrics".
<P>

<P>
<HR>

<P>
<!-- ======== NESTED CLASS SUMMARY ======== -->

<A NAME="nested_class_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Nested Class Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>static&nbsp;class</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#BAD_CYCLES">BAD_CYCLES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of instrument cycles in which 80% or more of base calls were no-calls.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;<A HREF="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</A></CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#CATEGORY">CATEGORY</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;One of either UNPAIRED (for a fragment run), FIRST_OF_PAIR when metrics are for only the
 first read in a paired run, SECOND_OF_PAIR when the metrics are for only the second read
 in a paired run or PAIR when the metrics are aggregated for both first and second reads
 in a pair.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#MEAN_READ_LENGTH">MEAN_READ_LENGTH</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The mean read length of the set of reads examined.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_ADAPTER">PCT_ADAPTER</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of PF reads that are unaligned and match to a known adapter sequence right from the
 start of the read.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_CHIMERAS">PCT_CHIMERAS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of reads that map outside of a maximum insert size (usually 100kb) or that have
 the two ends mapping to different chromosomes.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS">PCT_PF_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of reads that are PF (PF_READS / TOTAL_READS)</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS_ALIGNED">PCT_PF_READS_ALIGNED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of PF reads that aligned to the reference sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_READS_ALIGNED_IN_PAIRS">PCT_READS_ALIGNED_IN_PAIRS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of reads whose mate pair was also aligned to the reference.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_ALIGNED_BASES">PF_ALIGNED_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The total number of aligned bases, in all mapped PF reads, that are aligned to the reference sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_BASES">PF_HQ_ALIGNED_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of bases aligned to the reference sequence in reads that were mapped at high
 quality.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_Q20_BASES">PF_HQ_ALIGNED_Q20_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The subset of PF_HQ_ALIGNED_BASES where the base call quality was Q20 or higher.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_READS">PF_HQ_ALIGNED_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF reads that were aligned to the reference sequence with a mapping quality of
 Q20 or higher signifying that the aligner estimates a 1/100 (or smaller) chance that the
 alignment is wrong.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ERROR_RATE">PF_HQ_ERROR_RATE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of bases that mismatch the reference in PF HQ aligned reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_MEDIAN_MISMATCHES">PF_HQ_MEDIAN_MISMATCHES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The median number of mismatches versus the reference sequence in reads that were aligned
 to the reference at high quality (i.e.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_INDEL_RATE">PF_INDEL_RATE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of insertion and deletion events per 100 aligned bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_MISMATCH_RATE">PF_MISMATCH_RATE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The rate of bases mismatching the reference for all bases aligned to the reference sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_NOISE_READS">PF_NOISE_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF reads that are marked as noise reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_READS">PF_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF reads where PF is defined as passing Illumina's filter.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#PF_READS_ALIGNED">PF_READS_ALIGNED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF reads that were aligned to the reference sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#READS_ALIGNED_IN_PAIRS">READS_ALIGNED_IN_PAIRS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of aligned reads whose mate pair was also aligned to the reference.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#STRAND_BALANCE">STRAND_BALANCE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of PF reads aligned to the positive strand of the genome divided by the number of
 PF reads aligned to the genome.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#TOTAL_READS">TOTAL_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The total number of reads including all PF and non-PF reads.</TD>
</TR>
</TABLE>
&nbsp;<A NAME="fields_inherited_from_class_picard.metrics.MultilevelMetrics"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Fields inherited from class picard.metrics.<A HREF="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../picard/metrics/MultilevelMetrics.html#LIBRARY">LIBRARY</A>, <A HREF="../../picard/metrics/MultilevelMetrics.html#READ_GROUP">READ_GROUP</A>, <A HREF="../../picard/metrics/MultilevelMetrics.html#SAMPLE">SAMPLE</A></CODE></TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../picard/analysis/AlignmentSummaryMetrics.html#AlignmentSummaryMetrics()">AlignmentSummaryMetrics</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class htsjdk.samtools.metrics.MetricBase</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>equals, hashCode, toString</CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ============ FIELD DETAIL =========== -->

<A NAME="field_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Field Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="CATEGORY"><!-- --></A><H3>
CATEGORY</H3>
<PRE>
public <A HREF="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</A> <B>CATEGORY</B></PRE>
<DL>
<DD>One of either UNPAIRED (for a fragment run), FIRST_OF_PAIR when metrics are for only the
 first read in a paired run, SECOND_OF_PAIR when the metrics are for only the second read
 in a paired run or PAIR when the metrics are aggregated for both first and second reads
 in a pair.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="TOTAL_READS"><!-- --></A><H3>
TOTAL_READS</H3>
<PRE>
public long <B>TOTAL_READS</B></PRE>
<DL>
<DD>The total number of reads including all PF and non-PF reads. When CATEGORY equals PAIR
 this value will be 2x the number of clusters.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_READS"><!-- --></A><H3>
PF_READS</H3>
<PRE>
public long <B>PF_READS</B></PRE>
<DL>
<DD>The number of PF reads where PF is defined as passing Illumina's filter.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_PF_READS"><!-- --></A><H3>
PCT_PF_READS</H3>
<PRE>
public double <B>PCT_PF_READS</B></PRE>
<DL>
<DD>The percentage of reads that are PF (PF_READS / TOTAL_READS)
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_NOISE_READS"><!-- --></A><H3>
PF_NOISE_READS</H3>
<PRE>
public long <B>PF_NOISE_READS</B></PRE>
<DL>
<DD>The number of PF reads that are marked as noise reads.  A noise read is one which is composed
 entirely of A bases and/or N bases. These reads are marked as they are usually artifactual and
 are of no use in downstream analysis.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_READS_ALIGNED"><!-- --></A><H3>
PF_READS_ALIGNED</H3>
<PRE>
public long <B>PF_READS_ALIGNED</B></PRE>
<DL>
<DD>The number of PF reads that were aligned to the reference sequence. This includes reads that
 aligned with low quality (i.e. their alignments are ambiguous).
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_PF_READS_ALIGNED"><!-- --></A><H3>
PCT_PF_READS_ALIGNED</H3>
<PRE>
public double <B>PCT_PF_READS_ALIGNED</B></PRE>
<DL>
<DD>The percentage of PF reads that aligned to the reference sequence. PF_READS_ALIGNED / PF_READS
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_ALIGNED_BASES"><!-- --></A><H3>
PF_ALIGNED_BASES</H3>
<PRE>
public long <B>PF_ALIGNED_BASES</B></PRE>
<DL>
<DD>The total number of aligned bases, in all mapped PF reads, that are aligned to the reference sequence.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_HQ_ALIGNED_READS"><!-- --></A><H3>
PF_HQ_ALIGNED_READS</H3>
<PRE>
public long <B>PF_HQ_ALIGNED_READS</B></PRE>
<DL>
<DD>The number of PF reads that were aligned to the reference sequence with a mapping quality of
 Q20 or higher signifying that the aligner estimates a 1/100 (or smaller) chance that the
 alignment is wrong.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_HQ_ALIGNED_BASES"><!-- --></A><H3>
PF_HQ_ALIGNED_BASES</H3>
<PRE>
public long <B>PF_HQ_ALIGNED_BASES</B></PRE>
<DL>
<DD>The number of bases aligned to the reference sequence in reads that were mapped at high
 quality.  Will usually approximate PF_HQ_ALIGNED_READS * READ_LENGTH but may differ when
 either mixed read lengths are present or many reads are aligned with gaps.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_HQ_ALIGNED_Q20_BASES"><!-- --></A><H3>
PF_HQ_ALIGNED_Q20_BASES</H3>
<PRE>
public long <B>PF_HQ_ALIGNED_Q20_BASES</B></PRE>
<DL>
<DD>The subset of PF_HQ_ALIGNED_BASES where the base call quality was Q20 or higher.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_HQ_MEDIAN_MISMATCHES"><!-- --></A><H3>
PF_HQ_MEDIAN_MISMATCHES</H3>
<PRE>
public double <B>PF_HQ_MEDIAN_MISMATCHES</B></PRE>
<DL>
<DD>The median number of mismatches versus the reference sequence in reads that were aligned
 to the reference at high quality (i.e. PF_HQ_ALIGNED READS).
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_MISMATCH_RATE"><!-- --></A><H3>
PF_MISMATCH_RATE</H3>
<PRE>
public double <B>PF_MISMATCH_RATE</B></PRE>
<DL>
<DD>The rate of bases mismatching the reference for all bases aligned to the reference sequence.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_HQ_ERROR_RATE"><!-- --></A><H3>
PF_HQ_ERROR_RATE</H3>
<PRE>
public double <B>PF_HQ_ERROR_RATE</B></PRE>
<DL>
<DD>The percentage of bases that mismatch the reference in PF HQ aligned reads.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_INDEL_RATE"><!-- --></A><H3>
PF_INDEL_RATE</H3>
<PRE>
public double <B>PF_INDEL_RATE</B></PRE>
<DL>
<DD>The number of insertion and deletion events per 100 aligned bases.  Uses the number of events
 as the numerator, not the number of inserted or deleted bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEAN_READ_LENGTH"><!-- --></A><H3>
MEAN_READ_LENGTH</H3>
<PRE>
public double <B>MEAN_READ_LENGTH</B></PRE>
<DL>
<DD>The mean read length of the set of reads examined.  When looking at the data for a single lane with
 equal length reads this number is just the read length.  When looking at data for merged lanes with
 differing read lengths this is the mean read length of all reads.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="READS_ALIGNED_IN_PAIRS"><!-- --></A><H3>
READS_ALIGNED_IN_PAIRS</H3>
<PRE>
public long <B>READS_ALIGNED_IN_PAIRS</B></PRE>
<DL>
<DD>The number of aligned reads whose mate pair was also aligned to the reference.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_READS_ALIGNED_IN_PAIRS"><!-- --></A><H3>
PCT_READS_ALIGNED_IN_PAIRS</H3>
<PRE>
public double <B>PCT_READS_ALIGNED_IN_PAIRS</B></PRE>
<DL>
<DD>The percentage of reads whose mate pair was also aligned to the reference.
 READS_ALIGNED_IN_PAIRS / PF_READS_ALIGNED
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="BAD_CYCLES"><!-- --></A><H3>
BAD_CYCLES</H3>
<PRE>
public long <B>BAD_CYCLES</B></PRE>
<DL>
<DD>The number of instrument cycles in which 80% or more of base calls were no-calls.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="STRAND_BALANCE"><!-- --></A><H3>
STRAND_BALANCE</H3>
<PRE>
public double <B>STRAND_BALANCE</B></PRE>
<DL>
<DD>The number of PF reads aligned to the positive strand of the genome divided by the number of
 PF reads aligned to the genome.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_CHIMERAS"><!-- --></A><H3>
PCT_CHIMERAS</H3>
<PRE>
public double <B>PCT_CHIMERAS</B></PRE>
<DL>
<DD>The percentage of reads that map outside of a maximum insert size (usually 100kb) or that have
 the two ends mapping to different chromosomes.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_ADAPTER"><!-- --></A><H3>
PCT_ADAPTER</H3>
<PRE>
public double <B>PCT_ADAPTER</B></PRE>
<DL>
<DD>The percentage of PF reads that are unaligned and match to a known adapter sequence right from the
 start of the read.
<P>
<DL>
</DL>
</DL>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="AlignmentSummaryMetrics()"><!-- --></A><H3>
AlignmentSummaryMetrics</H3>
<PRE>
public <B>AlignmentSummaryMetrics</B>()</PRE>
<DL>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/AlignmentSummaryMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;PREV CLASS&nbsp;
&nbsp;<A HREF="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/analysis/AlignmentSummaryMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="AlignmentSummaryMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;<A HREF="#nested_class_summary">NESTED</A>&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
