<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:41 EST 2015 -->
<TITLE>
CollectWgsMetrics.WgsMetrics (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="CollectWgsMetrics.WgsMetrics (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/CollectWgsMetrics.WgsMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/analysis/CollectWgsMetricsFromQuerySorted.html" title="class in picard.analysis"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/analysis/CollectWgsMetrics.WgsMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="CollectWgsMetrics.WgsMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.analysis</FONT>
<BR>
Class CollectWgsMetrics.WgsMetrics</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../resources/inherit.gif" ALT="extended by ">htsjdk.samtools.metrics.MetricBase
      <IMG SRC="../../resources/inherit.gif" ALT="extended by "><B>picard.analysis.CollectWgsMetrics.WgsMetrics</B>
</PRE>
<DL>
<DT><B>Direct Known Subclasses:</B> <DD><A HREF="../../picard/analysis/CollectRawWgsMetrics.RawWgsMetrics.html" title="class in picard.analysis">CollectRawWgsMetrics.RawWgsMetrics</A>, <A HREF="../../picard/analysis/CollectWgsMetricsFromQuerySorted.QuerySortedSeqMetrics.html" title="class in picard.analysis">CollectWgsMetricsFromQuerySorted.QuerySortedSeqMetrics</A>, <A HREF="../../picard/analysis/CollectWgsMetricsFromSampledSites.SampledWgsMetrics.html" title="class in picard.analysis">CollectWgsMetricsFromSampledSites.SampledWgsMetrics</A></DD>
</DL>
<DL>
<DT><B>Enclosing class:</B><DD><A HREF="../../picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis">CollectWgsMetrics</A></DD>
</DL>
<HR>
<DL>
<DT><PRE>public static class <B>CollectWgsMetrics.WgsMetrics</B><DT>extends htsjdk.samtools.metrics.MetricBase</DL>
</PRE>

<P>
Metrics for evaluating the performance of whole genome sequencing experiments.
<P>

<P>
<HR>

<P>
<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#GENOME_TERRITORY">GENOME_TERRITORY</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The number of non-N bases in the genome reference over which coverage will be evaluated.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#MAD_COVERAGE">MAD_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The median absolute deviation of coverage of the genome after all filters are applied.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#MEAN_COVERAGE">MEAN_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The mean coverage in bases of the genome territory, after all filters are applied.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#MEDIAN_COVERAGE">MEDIAN_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The median coverage in bases of the genome territory, after all filters are applied.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_100X">PCT_100X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 100X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_10X">PCT_10X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 10X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_15X">PCT_15X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 15X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_20X">PCT_20X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 20X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_25X">PCT_25X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 25X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_30X">PCT_30X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 30X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_40X">PCT_40X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 40X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_50X">PCT_50X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 50X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_5X">PCT_5X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 5X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_60X">PCT_60X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 60X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_70X">PCT_70X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 70X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_80X">PCT_80X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 80X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_90X">PCT_90X</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of bases that attained at least 90X sequence coverage in post-filtering bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_BASEQ">PCT_EXC_BASEQ</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of aligned bases that were filtered out because they were of low base quality (default is < 20).</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_CAPPED">PCT_EXC_CAPPED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of aligned bases that were filtered out because they would have raised coverage above the capped value (default cap = 250x).</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_DUPE">PCT_EXC_DUPE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_MAPQ">PCT_EXC_MAPQ</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of aligned bases that were filtered out because they were in reads with low mapping quality (default is < 20).</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_OVERLAP">PCT_EXC_OVERLAP</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of aligned bases that were filtered out because they were the second observation from an insert with overlapping reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_TOTAL">PCT_EXC_TOTAL</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The total fraction of aligned bases excluded due to all filters.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#PCT_EXC_UNPAIRED">PCT_EXC_UNPAIRED</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The fraction of aligned bases that were filtered out because they were in reads without a mapped mate pair.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#SD_COVERAGE">SD_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The standard deviation of coverage of the genome after all filters are applied.</TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../picard/analysis/CollectWgsMetrics.WgsMetrics.html#CollectWgsMetrics.WgsMetrics()">CollectWgsMetrics.WgsMetrics</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class htsjdk.samtools.metrics.MetricBase</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>equals, hashCode, toString</CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ============ FIELD DETAIL =========== -->

<A NAME="field_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Field Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="GENOME_TERRITORY"><!-- --></A><H3>
GENOME_TERRITORY</H3>
<PRE>
public long <B>GENOME_TERRITORY</B></PRE>
<DL>
<DD>The number of non-N bases in the genome reference over which coverage will be evaluated.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEAN_COVERAGE"><!-- --></A><H3>
MEAN_COVERAGE</H3>
<PRE>
public double <B>MEAN_COVERAGE</B></PRE>
<DL>
<DD>The mean coverage in bases of the genome territory, after all filters are applied.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="SD_COVERAGE"><!-- --></A><H3>
SD_COVERAGE</H3>
<PRE>
public double <B>SD_COVERAGE</B></PRE>
<DL>
<DD>The standard deviation of coverage of the genome after all filters are applied.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEDIAN_COVERAGE"><!-- --></A><H3>
MEDIAN_COVERAGE</H3>
<PRE>
public double <B>MEDIAN_COVERAGE</B></PRE>
<DL>
<DD>The median coverage in bases of the genome territory, after all filters are applied.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MAD_COVERAGE"><!-- --></A><H3>
MAD_COVERAGE</H3>
<PRE>
public double <B>MAD_COVERAGE</B></PRE>
<DL>
<DD>The median absolute deviation of coverage of the genome after all filters are applied.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_MAPQ"><!-- --></A><H3>
PCT_EXC_MAPQ</H3>
<PRE>
public double <B>PCT_EXC_MAPQ</B></PRE>
<DL>
<DD>The fraction of aligned bases that were filtered out because they were in reads with low mapping quality (default is < 20).
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_DUPE"><!-- --></A><H3>
PCT_EXC_DUPE</H3>
<PRE>
public double <B>PCT_EXC_DUPE</B></PRE>
<DL>
<DD>The fraction of aligned bases that were filtered out because they were in reads marked as duplicates.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_UNPAIRED"><!-- --></A><H3>
PCT_EXC_UNPAIRED</H3>
<PRE>
public double <B>PCT_EXC_UNPAIRED</B></PRE>
<DL>
<DD>The fraction of aligned bases that were filtered out because they were in reads without a mapped mate pair.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_BASEQ"><!-- --></A><H3>
PCT_EXC_BASEQ</H3>
<PRE>
public double <B>PCT_EXC_BASEQ</B></PRE>
<DL>
<DD>The fraction of aligned bases that were filtered out because they were of low base quality (default is < 20).
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_OVERLAP"><!-- --></A><H3>
PCT_EXC_OVERLAP</H3>
<PRE>
public double <B>PCT_EXC_OVERLAP</B></PRE>
<DL>
<DD>The fraction of aligned bases that were filtered out because they were the second observation from an insert with overlapping reads.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_CAPPED"><!-- --></A><H3>
PCT_EXC_CAPPED</H3>
<PRE>
public double <B>PCT_EXC_CAPPED</B></PRE>
<DL>
<DD>The fraction of aligned bases that were filtered out because they would have raised coverage above the capped value (default cap = 250x).
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_EXC_TOTAL"><!-- --></A><H3>
PCT_EXC_TOTAL</H3>
<PRE>
public double <B>PCT_EXC_TOTAL</B></PRE>
<DL>
<DD>The total fraction of aligned bases excluded due to all filters.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_5X"><!-- --></A><H3>
PCT_5X</H3>
<PRE>
public double <B>PCT_5X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 5X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_10X"><!-- --></A><H3>
PCT_10X</H3>
<PRE>
public double <B>PCT_10X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 10X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_15X"><!-- --></A><H3>
PCT_15X</H3>
<PRE>
public double <B>PCT_15X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 15X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_20X"><!-- --></A><H3>
PCT_20X</H3>
<PRE>
public double <B>PCT_20X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 20X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_25X"><!-- --></A><H3>
PCT_25X</H3>
<PRE>
public double <B>PCT_25X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 25X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_30X"><!-- --></A><H3>
PCT_30X</H3>
<PRE>
public double <B>PCT_30X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 30X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_40X"><!-- --></A><H3>
PCT_40X</H3>
<PRE>
public double <B>PCT_40X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 40X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_50X"><!-- --></A><H3>
PCT_50X</H3>
<PRE>
public double <B>PCT_50X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 50X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_60X"><!-- --></A><H3>
PCT_60X</H3>
<PRE>
public double <B>PCT_60X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 60X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_70X"><!-- --></A><H3>
PCT_70X</H3>
<PRE>
public double <B>PCT_70X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 70X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_80X"><!-- --></A><H3>
PCT_80X</H3>
<PRE>
public double <B>PCT_80X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 80X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_90X"><!-- --></A><H3>
PCT_90X</H3>
<PRE>
public double <B>PCT_90X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 90X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_100X"><!-- --></A><H3>
PCT_100X</H3>
<PRE>
public double <B>PCT_100X</B></PRE>
<DL>
<DD>The fraction of bases that attained at least 100X sequence coverage in post-filtering bases.
<P>
<DL>
</DL>
</DL>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="CollectWgsMetrics.WgsMetrics()"><!-- --></A><H3>
CollectWgsMetrics.WgsMetrics</H3>
<PRE>
public <B>CollectWgsMetrics.WgsMetrics</B>()</PRE>
<DL>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/CollectWgsMetrics.WgsMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/analysis/CollectWgsMetrics.html" title="class in picard.analysis"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/analysis/CollectWgsMetricsFromQuerySorted.html" title="class in picard.analysis"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/analysis/CollectWgsMetrics.WgsMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="CollectWgsMetrics.WgsMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
