<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_35) on Mon Nov 16 12:51:41 EST 2015 -->
<TITLE>
RnaSeqMetrics (PICARD JDK API Documentation)
</TITLE>

<META NAME="date" CONTENT="2015-11-16">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="RnaSeqMetrics (PICARD JDK API Documentation)";
    }
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">
<HR>


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/RnaSeqMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/analysis/RnaSeqMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="RnaSeqMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
picard.analysis</FONT>
<BR>
Class RnaSeqMetrics</H2>
<PRE>
java.lang.Object
  <IMG SRC="../../resources/inherit.gif" ALT="extended by ">htsjdk.samtools.metrics.MetricBase
      <IMG SRC="../../resources/inherit.gif" ALT="extended by "><A HREF="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">picard.metrics.MultilevelMetrics</A>
          <IMG SRC="../../resources/inherit.gif" ALT="extended by "><B>picard.analysis.RnaSeqMetrics</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>RnaSeqMetrics</B><DT>extends <A HREF="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</A></DL>
</PRE>

<P>
Metrics about the alignment of RNA-seq reads within a SAM file to genes, produced by the CollectRnaSeqMetrics
 program and usually stored in a file with the extension ".rna_metrics".
<P>

<P>
<HR>

<P>
<!-- =========== FIELD SUMMARY =========== -->

<A NAME="field_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Field Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#CODING_BASES">CODING_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of bases in primary aligments that align to a non-UTR coding base for some gene, and not ribosomal sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#CORRECT_STRAND_READS">CORRECT_STRAND_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of aligned reads that map to the correct strand.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#IGNORED_READS">IGNORED_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of primary alignments that map to a sequence specified on command-line as IGNORED_SEQUENCE.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#INCORRECT_STRAND_READS">INCORRECT_STRAND_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of aligned reads that map to the incorrect strand.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#INTERGENIC_BASES">INTERGENIC_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of bases in primary aligments that do not align to any gene.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#INTRONIC_BASES">INTRONIC_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of bases in primary aligments that align to an intronic base for some gene, and not a coding or UTR base.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#MEDIAN_3PRIME_BIAS">MEDIAN_3PRIME_BIAS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The median 3 prime bias of the 1000 most highly expressed transcripts, where 3 prime bias is calculated per
 transcript as: mean coverage of the 3' most 100 bases divided by the mean coverage of the whole transcript.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#MEDIAN_5PRIME_BIAS">MEDIAN_5PRIME_BIAS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The median 5 prime bias of the 1000 most highly expressed transcripts, where 5 prime bias is calculated per
 transcript as: mean coverage of the 5' most 100 bases divided by the mean coverage of the whole transcript.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#MEDIAN_5PRIME_TO_3PRIME_BIAS">MEDIAN_5PRIME_TO_3PRIME_BIAS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The ratio of coverage at the 5' end of to the 3' end based on the 1000 most highly expressed transcripts.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#MEDIAN_CV_COVERAGE">MEDIAN_CV_COVERAGE</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The median CV of coverage of the 1000 most highly expressed transcripts.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_CODING_BASES">PCT_CODING_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CODING_BASES / PF_ALIGNED_BASES</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_CORRECT_STRAND_READS">PCT_CORRECT_STRAND_READS</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;CORRECT_STRAND_READS/(CORRECT_STRAND_READS + INCORRECT_STRAND_READS).</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_INTERGENIC_BASES">PCT_INTERGENIC_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;INTERGENIC_BASES / PF_ALIGNED_BASES</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_INTRONIC_BASES">PCT_INTRONIC_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;INTRONIC_BASES / PF_ALIGNED_BASES</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_MRNA_BASES">PCT_MRNA_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;PCT_UTR_BASES + PCT_CODING_BASES</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.lang.Double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_RIBOSOMAL_BASES">PCT_RIBOSOMAL_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;RIBOSOMAL_BASES / PF_ALIGNED_BASES</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_USABLE_BASES">PCT_USABLE_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The percentage of bases mapping to mRNA divided by the total number of PF bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;double</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PCT_UTR_BASES">PCT_UTR_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;UTR_BASES / PF_ALIGNED_BASES</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PF_ALIGNED_BASES">PF_ALIGNED_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The total number of aligned PF bases.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#PF_BASES">PF_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The total number of PF bases including non-aligned reads.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;java.lang.Long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#RIBOSOMAL_BASES">RIBOSOMAL_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of bases in primary aligments that align to ribosomal sequence.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;long</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#UTR_BASES">UTR_BASES</A></B></CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Number of bases in primary aligments that align to a UTR base for some gene, and not a coding base.</TD>
</TR>
</TABLE>
&nbsp;<A NAME="fields_inherited_from_class_picard.metrics.MultilevelMetrics"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Fields inherited from class picard.metrics.<A HREF="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../picard/metrics/MultilevelMetrics.html#LIBRARY">LIBRARY</A>, <A HREF="../../picard/metrics/MultilevelMetrics.html#READ_GROUP">READ_GROUP</A>, <A HREF="../../picard/metrics/MultilevelMetrics.html#SAMPLE">SAMPLE</A></CODE></TD>
</TR>
</TABLE>
&nbsp;
<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../picard/analysis/RnaSeqMetrics.html#RnaSeqMetrics()">RnaSeqMetrics</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class htsjdk.samtools.metrics.MetricBase</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>equals, hashCode, toString</CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.Object</B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ============ FIELD DETAIL =========== -->

<A NAME="field_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Field Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="PF_BASES"><!-- --></A><H3>
PF_BASES</H3>
<PRE>
public long <B>PF_BASES</B></PRE>
<DL>
<DD>The total number of PF bases including non-aligned reads.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PF_ALIGNED_BASES"><!-- --></A><H3>
PF_ALIGNED_BASES</H3>
<PRE>
public long <B>PF_ALIGNED_BASES</B></PRE>
<DL>
<DD>The total number of aligned PF bases.  Non-primary alignments are not counted. Bases in aligned reads that
 do not correspond to reference (e.g. soft clips, insertions) are not counted.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="RIBOSOMAL_BASES"><!-- --></A><H3>
RIBOSOMAL_BASES</H3>
<PRE>
public java.lang.Long <B>RIBOSOMAL_BASES</B></PRE>
<DL>
<DD>Number of bases in primary aligments that align to ribosomal sequence.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="CODING_BASES"><!-- --></A><H3>
CODING_BASES</H3>
<PRE>
public long <B>CODING_BASES</B></PRE>
<DL>
<DD>Number of bases in primary aligments that align to a non-UTR coding base for some gene, and not ribosomal sequence.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="UTR_BASES"><!-- --></A><H3>
UTR_BASES</H3>
<PRE>
public long <B>UTR_BASES</B></PRE>
<DL>
<DD>Number of bases in primary aligments that align to a UTR base for some gene, and not a coding base.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="INTRONIC_BASES"><!-- --></A><H3>
INTRONIC_BASES</H3>
<PRE>
public long <B>INTRONIC_BASES</B></PRE>
<DL>
<DD>Number of bases in primary aligments that align to an intronic base for some gene, and not a coding or UTR base.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="INTERGENIC_BASES"><!-- --></A><H3>
INTERGENIC_BASES</H3>
<PRE>
public long <B>INTERGENIC_BASES</B></PRE>
<DL>
<DD>Number of bases in primary aligments that do not align to any gene.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="IGNORED_READS"><!-- --></A><H3>
IGNORED_READS</H3>
<PRE>
public long <B>IGNORED_READS</B></PRE>
<DL>
<DD>Number of primary alignments that map to a sequence specified on command-line as IGNORED_SEQUENCE.  These are not
 counted in PF_ALIGNED_BASES, CORRECT_STRAND_READS, INCORRECT_STRAND_READS, or any of the base-counting metrics.
 These reads are counted in PF_BASES.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="CORRECT_STRAND_READS"><!-- --></A><H3>
CORRECT_STRAND_READS</H3>
<PRE>
public long <B>CORRECT_STRAND_READS</B></PRE>
<DL>
<DD>Number of aligned reads that map to the correct strand.  0 if library is not strand-specific.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="INCORRECT_STRAND_READS"><!-- --></A><H3>
INCORRECT_STRAND_READS</H3>
<PRE>
public long <B>INCORRECT_STRAND_READS</B></PRE>
<DL>
<DD>Number of aligned reads that map to the incorrect strand.  0 if library is not strand-specific.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_RIBOSOMAL_BASES"><!-- --></A><H3>
PCT_RIBOSOMAL_BASES</H3>
<PRE>
public java.lang.Double <B>PCT_RIBOSOMAL_BASES</B></PRE>
<DL>
<DD>RIBOSOMAL_BASES / PF_ALIGNED_BASES
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_CODING_BASES"><!-- --></A><H3>
PCT_CODING_BASES</H3>
<PRE>
public double <B>PCT_CODING_BASES</B></PRE>
<DL>
<DD>CODING_BASES / PF_ALIGNED_BASES
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_UTR_BASES"><!-- --></A><H3>
PCT_UTR_BASES</H3>
<PRE>
public double <B>PCT_UTR_BASES</B></PRE>
<DL>
<DD>UTR_BASES / PF_ALIGNED_BASES
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_INTRONIC_BASES"><!-- --></A><H3>
PCT_INTRONIC_BASES</H3>
<PRE>
public double <B>PCT_INTRONIC_BASES</B></PRE>
<DL>
<DD>INTRONIC_BASES / PF_ALIGNED_BASES
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_INTERGENIC_BASES"><!-- --></A><H3>
PCT_INTERGENIC_BASES</H3>
<PRE>
public double <B>PCT_INTERGENIC_BASES</B></PRE>
<DL>
<DD>INTERGENIC_BASES / PF_ALIGNED_BASES
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_MRNA_BASES"><!-- --></A><H3>
PCT_MRNA_BASES</H3>
<PRE>
public double <B>PCT_MRNA_BASES</B></PRE>
<DL>
<DD>PCT_UTR_BASES + PCT_CODING_BASES
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_USABLE_BASES"><!-- --></A><H3>
PCT_USABLE_BASES</H3>
<PRE>
public double <B>PCT_USABLE_BASES</B></PRE>
<DL>
<DD>The percentage of bases mapping to mRNA divided by the total number of PF bases.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="PCT_CORRECT_STRAND_READS"><!-- --></A><H3>
PCT_CORRECT_STRAND_READS</H3>
<PRE>
public double <B>PCT_CORRECT_STRAND_READS</B></PRE>
<DL>
<DD>CORRECT_STRAND_READS/(CORRECT_STRAND_READS + INCORRECT_STRAND_READS).  0 if library is not strand-specific.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEDIAN_CV_COVERAGE"><!-- --></A><H3>
MEDIAN_CV_COVERAGE</H3>
<PRE>
public double <B>MEDIAN_CV_COVERAGE</B></PRE>
<DL>
<DD>The median CV of coverage of the 1000 most highly expressed transcripts. Ideal value = 0.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEDIAN_5PRIME_BIAS"><!-- --></A><H3>
MEDIAN_5PRIME_BIAS</H3>
<PRE>
public double <B>MEDIAN_5PRIME_BIAS</B></PRE>
<DL>
<DD>The median 5 prime bias of the 1000 most highly expressed transcripts, where 5 prime bias is calculated per
 transcript as: mean coverage of the 5' most 100 bases divided by the mean coverage of the whole transcript.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEDIAN_3PRIME_BIAS"><!-- --></A><H3>
MEDIAN_3PRIME_BIAS</H3>
<PRE>
public double <B>MEDIAN_3PRIME_BIAS</B></PRE>
<DL>
<DD>The median 3 prime bias of the 1000 most highly expressed transcripts, where 3 prime bias is calculated per
 transcript as: mean coverage of the 3' most 100 bases divided by the mean coverage of the whole transcript.
<P>
<DL>
</DL>
</DL>
<HR>

<A NAME="MEDIAN_5PRIME_TO_3PRIME_BIAS"><!-- --></A><H3>
MEDIAN_5PRIME_TO_3PRIME_BIAS</H3>
<PRE>
public double <B>MEDIAN_5PRIME_TO_3PRIME_BIAS</B></PRE>
<DL>
<DD>The ratio of coverage at the 5' end of to the 3' end based on the 1000 most highly expressed transcripts.
<P>
<DL>
</DL>
</DL>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="RnaSeqMetrics()"><!-- --></A><H3>
RnaSeqMetrics</H3>
<PRE>
public <B>RnaSeqMetrics</B>()</PRE>
<DL>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/RnaSeqMetrics.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../index-all.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../picard/analysis/QualityScoreDistribution.html" title="class in picard.analysis"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../picard/analysis/RrbsCpgDetailMetrics.html" title="class in picard.analysis"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../index.html?picard/analysis/RnaSeqMetrics.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="RnaSeqMetrics.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;<A HREF="#field_summary">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#methods_inherited_from_class_htsjdk.samtools.metrics.MetricBase">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;<A HREF="#field_detail">FIELD</A>&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;METHOD</FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>

</BODY>
</HTML>
